<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.0.37">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Eliot McKinley - Scientific Research</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
span.underline{text-decoration: underline;}
div.column{display: inline-block; vertical-align: top; width: 50%;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./favicon.png" rel="icon" type="image/png">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.6/require.min.js" integrity="sha512-c3Nl8+7g4LMSTdrm621y7kf9v3SDPnhxLNhcjFJbKECVnmZHTdo+IRO05sNLTH/D3vA6u1X32ehoLC7WFVdheg==" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" crossorigin="anonymous"></script>
<script type="application/javascript">define('jquery', [],function() {return window.jQuery;})</script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">Eliot McKinley</span>
  </a>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./about.html">About</a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="./research.html" aria-current="page">Scientific Research</a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./soccer.html">Soccer</a>
  </li>  
</ul>
              <div id="quarto-search" class="" title="Search"></div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#published-papers" id="toc-published-papers" class="nav-link active" data-scroll-target="#published-papers">Published Papers</a>
  <ul class="collapse">
  <li><a href="#published-2022" id="toc-published-2022" class="nav-link" data-scroll-target="#published-2022">2022</a></li>
  <li><a href="#published-2021" id="toc-published-2021" class="nav-link" data-scroll-target="#published-2021">2021</a></li>
  <li><a href="#published-2020" id="toc-published-2020" class="nav-link" data-scroll-target="#published-2020">2020</a></li>
<<<<<<< HEAD
  <li><a href="#published-2019" id="toc-published-2019" class="nav-link" data-scroll-target="#published-2019">2019</a></li>
=======
>>>>>>> 9edb1aff55c5ced3256ae9d3470567a2ac9158f1
  <li><a href="#published-2018" id="toc-published-2018" class="nav-link" data-scroll-target="#published-2018">2018</a></li>
  <li><a href="#published-2017" id="toc-published-2017" class="nav-link" data-scroll-target="#published-2017">2017</a></li>
  <li><a href="#published-2016" id="toc-published-2016" class="nav-link" data-scroll-target="#published-2016">2016</a></li>
  <li><a href="#published-2015" id="toc-published-2015" class="nav-link" data-scroll-target="#published-2015">2015</a></li>
  <li><a href="#published-2014" id="toc-published-2014" class="nav-link" data-scroll-target="#published-2014">2014</a></li>
  <li><a href="#published-2013" id="toc-published-2013" class="nav-link" data-scroll-target="#published-2013">2013</a></li>
  <li><a href="#published-2012" id="toc-published-2012" class="nav-link" data-scroll-target="#published-2012">2012</a></li>
  <li><a href="#published-2011" id="toc-published-2011" class="nav-link" data-scroll-target="#published-2011">2011</a></li>
  <li><a href="#published-2010" id="toc-published-2010" class="nav-link" data-scroll-target="#published-2010">2010</a></li>
  <li><a href="#published-2009" id="toc-published-2009" class="nav-link" data-scroll-target="#published-2009">2009</a></li>
  <li><a href="#published-2008" id="toc-published-2008" class="nav-link" data-scroll-target="#published-2008">2008</a></li>
  </ul></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Scientific Research</h1>
</div>



<div class="quarto-title-meta">

    
    
  </div>
  

</header>

<h2 id="published-papers" class="anchored">Published Papers</h2>
<h3 id="published-2022" class="anchored">2022</h3>
<ul class="list-group list-group-flush"><li class="list-group-item"><strong>Eliot T. McKinley</strong>, Justin Shao, Samuel T. Ellis, Cody N. Heiser, Joseph T. Roland, Mary C. Macedonia, Paige N. Vega, Susie Shin, Robert J. Coffey, and Ken S. Lau. (2022) "MIRIAM: A machine and deep learning single-cell segmentation and quantification pipeline for multi-dimensional tissue images." <em>Cytometry</em><br><br><details><summary>Abstract</summary><ul>Increasingly, highly multiplexed tissue imaging methods are used to profile protein expression at the single-cell level. However, a critical limitation is the lack of robust cell segmentation tools for tissue sections. We present Multiplexed Image Resegmentation of Internal Aberrant Membranes (MIRIAM) that combines (a) a pipeline for cell segmentation and quantification that incorporates machine learning-based pixel classification to define cellular compartments, (b) a novel method for extending incomplete cell membranes, and (c) a deep learning-based cell shape descriptor. Using human colonic adenomas as an example, we show that MIRIAM is superior to widely utilized segmentation methods and provides a pipeline that is broadly applicable to different imaging platforms and tissue types.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://www.biorxiv.org/content/10.1101/790162v1" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-biorxiv" role="img" aria-label="Preprint"></i>
        Preprint
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://github.com/Coffey-Lab/MIRIAM" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-github" role="img" aria-label="Github"></i>
        Github
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/35084791/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1002/cyto.a.24541" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li>
<li class="list-group-item">Paige N. Vega, Avlant Nilsson, Manu P. Kumar, Hiroaki Niitsu, Alan J. Simmons, James Ro, Jiawei Wang, Zhengyi Chen, Brian A. Joughin, Wei Li, <strong>Eliot T. McKinley</strong>, Qi Liu, Joseph T. Roland, M. Kay Washington, Robert J. Coffey, Douglas A. Lauffenburger, and Ken S. Lau. (2022) "Cancer-Associated Fibroblasts and Squamous Epithelial Cells Constitute a Unique Microenvironment in a Mouse Model of Inflammation-Induced Colon Cancer." <em>Front Oncol</em><br><br><details><summary>Abstract</summary><ul>The tumor microenvironment plays a key role in the pathogenesis of colorectal tumors and contains various cell types including epithelial, immune, and mesenchymal cells. Characterization of the interactions between these cell types is necessary for revealing the complex nature of tumors. In this study, we used single-cell RNA-seq (scRNA-seq) to compare the tumor microenvironments between a mouse model of sporadic colorectal adenoma (Lrig1CreERT2/+;Apc2lox14/+) and a mouse model of inflammation-driven colorectal cancer induced by azoxymethane and dextran sodium sulfate (AOM/DSS). While both models develop tumors in the distal colon, we found that the two tumor types have distinct microenvironments. AOM/DSS tumors have an increased abundance of two populations of cancer-associated fibroblasts (CAFs) compared with APC tumors, and we revealed their divergent spatial association with tumor cells using multiplex immunofluorescence (MxIF) imaging. We also identified a unique squamous cell population in AOM/DSS tumors, whose origins were distinct from anal squamous epithelial cells. These cells were in higher proportions upon administration of a chemotherapy regimen of 5-Fluorouracil/Irinotecan. We used computational inference algorithms to predict cell-cell communication mediated by ligand-receptor interactions and downstream pathway activation, and identified potential mechanistic connections between CAFs and tumor cells, as well as CAFs and squamous epithelial cells. This study provides important preclinical insight into the microenvironment of two distinct models of colorectal tumors and reveals unique roles for CAFs and squamous epithelial cells in the AOM/DSS model of inflammation-driven cancer.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/35600339/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.3389/fonc.2022.878920" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li>
<<<<<<< HEAD
<li class="list-group-item">Coleman R. Harris, <strong>Eliot T. McKinley</strong>, Joseph T. Roland, Qi Liu, Martha J. Shrubsole, Ken S. Lau, Robert J. Coffey, Julia Wrobel, and Simon N. Vandekar. (2022) "Quantifying and correcting slide-to-slide variation in multiplexed immunofluorescence images." <em>Bioinformatics</em><br><br><details><summary>Abstract</summary><ul>Motivation: Multiplexed imaging is a nascent single-cell assay with a complex data structure susceptible to technical variability that disrupts inference. These in situ methods are valuable in understanding cell-cell interactions, but few standardized processing steps or normalization techniques of multiplexed imaging data are available.<br><br>Results: We implement and compare data transformations and normalization algorithms in multiplexed imaging data. Our methods adapt the ComBat and functional data registration methods to remove slide effects in this domain, and we present an evaluation framework to compare the proposed approaches. We present clear slide-to-slide variation in the raw, unadjusted data, and show that many of the proposed normalization methods reduce this variation while preserving and improving the biological signal. Further, we find that dividing multiplexed imaging data by its slide mean, and the functional data registration methods, perform the best under our proposed evaluation framework. In summary, this approach provides a foundation for better data quality and evaluation criteria in multiplexed imaging.<br><br>Availability: Source code is provided at: [https://github.com/statimagcoll/MultiplexedNormalization](https://github.com/statimagcoll/MultiplexedNormalization) and an R package to implement these methods is available here: [https://github.com/ColemanRHarris/mxnorm](https://github.com/ColemanRHarris/mxnorm).</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
=======
<li class="list-group-item">Coleman R Harris, <strong>Eliot T. McKinley</strong>, Joseph T Roland, Qi Liu, Martha J Shrubsole, Ken S Lau-, Robert J Coffey, Julia Wrobel, and Simon N Vandekar. (2022) "Quantifying and correcting slide-to-slide variation in multiplexed immunofluorescence images." <em>Bioinformatics</em><br><br><details><summary>Abstract</summary><ul>Motivation: Multiplexed imaging is a nascent single-cell assay with a complex data structure susceptible to technical variability that disrupts inference. These in situ methods are valuable in understanding cell-cell interactions, but few standardized processing steps or normalization techniques of multiplexed imaging data are available.<br><br>Results: We implement and compare data transformations and normalization algorithms in multiplexed imaging data. Our methods adapt the ComBat and functional data registration methods to remove slide effects in this domain, and we present an evaluation framework to compare the proposed approaches. We present clear slide-to-slide variation in the raw, unadjusted data, and show that many of the proposed normalization methods reduce this variation while preserving and improving the biological signal. Further, we find that dividing multiplexed imaging data by its slide mean, and the functional data registration methods, perform the best under our proposed evaluation framework. In summary, this approach provides a foundation for better data quality and evaluation criteria in multiplexed imaging.<br><br>Availability: Source code is provided at: [https://github.com/statimagcoll/MultiplexedNormalization](https://github.com/statimagcoll/MultiplexedNormalization) and an R package to implement these methods is available here: [https://github.com/ColemanRHarris/mxnorm](https://github.com/ColemanRHarris/mxnorm).</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
>>>>>>> 9edb1aff55c5ced3256ae9d3470567a2ac9158f1
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://www.biorxiv.org/content/10.1101/2021.07.16.452359v1" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-biorxiv" role="img" aria-label="Preprint"></i>
        Preprint
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://github.com/ColemanRHarris/mxnorm" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-github" role="img" aria-label="Github"></i>
        Github
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/34983062/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1093/bioinformatics/btab877" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li>
<<<<<<< HEAD
<li class="list-group-item">Denis Schapiro, Artem Sokolov, Clarence Yapp, Yu-An Chen, Jeremy L Muhlich, Joshua Hess, Allison L. Creason, Ajit J. Nirmal, Gregory J. Baker, Maulik K. Nariya, Jia-Ren Lin, Zoltan Maliga, Connor A. Jacobson, Matthew W. Hodgman, Juha Ruokonen, Samouil L. Farhi, Domenic Abbondanza, <strong>Eliot T. McKinley</strong>, Daniel Persson, Courtney Betts, Shamilene Sivagnanam, Aviv Regev, Jeremy Goecks, Robert J. Coffey, Lisa M. Coussens, Sandro Santagata, and Peter K. Sorger. (2022) "MCMICRO: a scalable, modular image-processing pipeline for multiplexed tissue imaging." <em>Nat Methods</em><br><br><details><summary>Abstract</summary><ul>Highly multiplexed tissue imaging makes detailed molecular analysis of single cells possible in a preserved spatial context. However, reproducible analysis of large multichannel images poses a substantial computational challenge. Here, we describe a modular and open-source computational pipeline, MCMICRO, for performing the sequential steps needed to transform whole-slide images into single-cell data. We demonstrate the use of MCMICRO on tissue and tumor images acquired using multiple imaging platforms, thereby providing a solid foundation for the continued development of tissue imaging software.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
=======
<li class="list-group-item">Denis Schapiro, Artem Sokolov, Clarence Yapp, Yu-An Chen, Jeremy L Muhlich, Joshua Hess, Allison L Creason, Ajit J Nirmal, Gregory J Baker, Maulik K Nariya, Jia-Ren Lin, Zoltan Maliga, Connor A Jacobson, Matthew W Hodgman, Juha Ruokonen, Samouil L Farhi, Domenic Abbondanza, <strong>Eliot T. McKinley</strong>, Daniel Persson, Courtney Betts, Shamilene Sivagnanam, Aviv Regev, Jeremy Goecks, Robert J Coffey, Lisa M Coussens, Sandro Santagata, and Peter K Sorger. (2022) "MCMICRO: a scalable, modular image-processing pipeline for multiplexed tissue imaging." <em>Nat Methods</em><br><br><details><summary>Abstract</summary><ul>Highly multiplexed tissue imaging makes detailed molecular analysis of single cells possible in a preserved spatial context. However, reproducible analysis of large multichannel images poses a substantial computational challenge. Here, we describe a modular and open-source computational pipeline, MCMICRO, for performing the sequential steps needed to transform whole-slide images into single-cell data. We demonstrate the use of MCMICRO on tissue and tumor images acquired using multiple imaging platforms, thereby providing a solid foundation for the continued development of tissue imaging software.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
>>>>>>> 9edb1aff55c5ced3256ae9d3470567a2ac9158f1
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://www.biorxiv.org/content/10.1101/2021.03.15.435473v1" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-biorxiv" role="img" aria-label="Preprint"></i>
        Preprint
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/34824477/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1038/s41592-021-01308-y" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li></ul>
<h3 id="published-2021" class="anchored">2021</h3>

<<<<<<< HEAD
<ul class="list-group list-group-flush"><li class="list-group-item">Qin Zhang, Dennis K. Jeppesen, James N. Higginbotham, Ramona Graves-Deal, Vincent Q. Trinh, Marisol A. Ramirez-Solano, Yoojin Sohn, Abigail C. Neininger, Nilay Taneja, <strong>Eliot T. McKinley</strong>, Hiroaki Niitsu, Zheng Cao, Rachel Evans, Sarah E. Glass, Kevin C. Ray, William H. Fissell, Salisha Hill, Kristie Lindsey Rose, Won Jae Huh, M. Kay Washington, Gregory D. Ayers, Dylan T. Burnette, Shivani Sharma, Leonard H. Rome, Jeffrey L. Franklin, Youngmin A. Lee, Qi Liu, and Robert J. Coffey. (2021) "Supermeres are functional extracellular nanoparticles replete with disease biomarkers and therapeutic targets." <em>Nat Cell Biol</em><br><br><details><summary>Abstract</summary><ul>Extracellular vesicles and exomere nanoparticles are under intense investigation as sources of clinically relevant cargo. Here we report the discovery of a distinct extracellular nanoparticle, termed supermere. Supermeres are morphologically distinct from exomeres and display a markedly greater uptake in vivo compared with small extracellular vesicles and exomeres. The protein and RNA composition of supermeres differs from small extracellular vesicles and exomeres. Supermeres are highly enriched with cargo involved in multiple cancers (glycolytic enzymes, TGFBI, miR-1246, MET, GPC1 and AGO2), Alzheimer's disease (APP) and cardiovascular disease (ACE2, ACE and PCSK9). The majority of extracellular RNA is associated with supermeres rather than small extracellular vesicles and exomeres. Cancer-derived supermeres increase lactate secretion, transfer cetuximab resistance and decrease hepatic lipids and glycogen in vivo. This study identifies a distinct functional nanoparticle replete with potential circulating biomarkers and therapeutic targets for a host of human diseases.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
=======
<ul class="list-group list-group-flush"><li class="list-group-item">Qin Zhang, Dennis K. Jeppesen, James N. Higginbotham, Ramona Graves-Deal, Vincent Q. Trinh, Marisol A. Ramirez, Yoojin Sohn, Abigail C. Neininger, Nilay Taneja, <strong>Eliot T. McKinley</strong>, Hiroaki Niitsu, Zheng Cao, Rachel Evans, Sarah E. Glass, Kevin C. Ray, William H. Fissell, Salisha Hill, Kristie Lindsey Rose, Won Jae Huh, Mary Kay Washington, Gregory Daniel Ayers, Dylan T. Burnette, Shivani Sharma, Leonard H. Rome, Jeffrey L. Franklin, Youngmin A. Lee, Qi Liu, and Robert J. Coffey. (2021) "Supermeres are functional extracellular nanoparticles replete with disease biomarkers and therapeutic targets." <em>Nat Cell Biol</em><br><br><details><summary>Abstract</summary><ul>Extracellular vesicles and exomere nanoparticles are under intense investigation as sources of clinically relevant cargo. Here we report the discovery of a distinct extracellular nanoparticle, termed supermere. Supermeres are morphologically distinct from exomeres and display a markedly greater uptake in vivo compared with small extracellular vesicles and exomeres. The protein and RNA composition of supermeres differs from small extracellular vesicles and exomeres. Supermeres are highly enriched with cargo involved in multiple cancers (glycolytic enzymes, TGFBI, miR-1246, MET, GPC1 and AGO2), Alzheimer's disease (APP) and cardiovascular disease (ACE2, ACE and PCSK9). The majority of extracellular RNA is associated with supermeres rather than small extracellular vesicles and exomeres. Cancer-derived supermeres increase lactate secretion, transfer cetuximab resistance and decrease hepatic lipids and glycogen in vivo. This study identifies a distinct functional nanoparticle replete with potential circulating biomarkers and therapeutic targets for a host of human diseases.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
>>>>>>> 9edb1aff55c5ced3256ae9d3470567a2ac9158f1
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/34887515/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1038/s41556-021-00805-8" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li>
<li class="list-group-item">Bhuminder Singh, Galina Bogatcheva, Evan Krystofiak, <strong>Eliot T. McKinley</strong>, Salisha Hill, Kristie Lindsey Rose, James N. Higginbotham, and Robert J. Coffey. (2021) "Induction of apically mistrafficked epiregulin disrupts epithelial polarity via aberrant EGFR signaling." <em>J Cell Sci</em><br><br><details><summary>Abstract</summary><ul>In polarized MDCK cells, disruption of the tyrosine-based YXXΦ basolateral trafficking motif (Y156A) in the epidermal growth factor receptor (EGFR) ligand epiregulin (EREG), results in its apical mistrafficking and transformation in vivo. However, the mechanisms underlying these dramatic effects are unknown. Using a doxycycline-inducible system in 3D Matrigel cultures, we now show that induction of Y156A EREG in fully formed MDCK cysts results in direct and complete delivery of mutant EREG to the apical cell surface. Within 3 days of induction, ectopic lumens were detected in mutant, but not wild-type, EREG-expressing cysts. Of note, these structures resembled histological features found in subcutaneous xenografts of mutant EREG-expressing MDCK cells. These ectopic lumens formed de novo rather than budding from the central lumen and depended on metalloprotease-mediated cleavage of EREG and subsequent EGFR activity. Moreover, the most frequent EREG mutation in human cancer (R147stop) resulted in its apical mistrafficking in engineered MDCK cells. Thus, induction of EREG apical mistrafficking is sufficient to disrupt selective aspects of polarity of a preformed polarized epithelium. This article has an associated First Person interview with the first author of the paper.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/34406412/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1242/jcs.255927" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li>
<li class="list-group-item">Junzhong Xu, Xiaoyu Jiang, Sean P. Devan, Lori R. Arlinghaus, <strong>Eliot T. McKinley</strong>, Jingping Xie, Zhongliang Zu, Qing Wang, A. Bapsi Chakravarthy, Yong Wang, and John C. Gore. (2021) "MRI-cytometry: Mapping nonparametric cell size distributions using diffusion MRI." <em>Magn Reson Med</em><br><br><details><summary>Abstract</summary><ul>Purpose: This report introduces and validates a new diffusion MRI-based method, termed MRI-cytometry, which can noninvasively map intravoxel, nonparametric cell size distributions in tissues.<br><br>Methods: MRI was used to acquire diffusion MRI signals with a range of diffusion times and gradient factors, and a model was fit to these data to derive estimates of cell size distributions. We implemented a 2-step fitting method to avoid noise-induced artificial peaks and provide reliable estimates of tumor cell size distributions. Computer simulations in silico, experimental measurements on cultured cells in vitro, and animal xenografts in vivo were used to validate the accuracy and precision of the method. Tumors in 7 patients with breast cancer were also imaged and analyzed using this MRI-cytometry approach on a clinical 3 Tesla MRI scanner.<br><br>Results: Simulations and experimental results confirm that MRI-cytometry can reliably map intravoxel, nonparametric cell size distributions and has the potential to discriminate smaller and larger cells. The application in breast cancer patients demonstrates the feasibility of direct translation of MRI-cytometry to clinical applications.<br><br>Conclusion: The proposed MRI-cytometry method can characterize nonparametric cell size distributions in human tumors, which potentially provides a practical imaging approach to derive specific histopathological information on biological tissues.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/32936478/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1002/mrm.28454" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li>
<<<<<<< HEAD
<li class="list-group-item">Bob Chen, Cherie' R. Scurrah, <strong>Eliot T. McKinley</strong>, Alan J. Simmons, Marisol A. Ramirez-Solano, Xiangzhu Zhu, Nicholas O. Markham, Cody N. Heiser, Paige N. Vega, Andrea Rolong, Hyeyon Kim, Quanhu Sheng, Julia L. Drewes, Yuan Zhou, Austin N. Southard-Smith, Yanwen Xu, James Ro, Angela L. Jones, Frank Revetta, Lynne D. Berry, Hiroaki Niitsu, Mirazul Islam, Karin Pelka, Matan Hofree, Jonathan H. Chen, Siranush Sarkizova, Kimmie Ng, Marios Giannakis, Genevieve M. Boland, Andrew J. Aguirre, Ana C. Anderson, Orit Rozenblatt-Rosen, Aviv Regev, Nir Hacohen, Kenta Kawasaki, Toshiro Sato, Jeremy A. Goettel, William M. Grady, Wei Zheng, M. Kay Washington, Qiuyin Cai, Cynthia L. Sears, James R. Goldenring, Jeffrey L. Franklin, Timothy Su, Won Jae Huh, Simon Vandekar, Joseph T. Roland, Qi Liu, Robert J. Coffey, Martha J. Shrubsole, and Ken S. Lau. (2021) "Differential pre-malignant programs and microenvironment chart distinct paths to malignancy in human colorectal polyps." <em>Cell</em><br><br><details><summary>Abstract</summary><ul>Colorectal cancers (CRCs) arise from precursor polyps whose cellular origins, molecular heterogeneity, and immunogenic potential may reveal diagnostic and therapeutic insights when analyzed at high resolution. We present a single-cell transcriptomic and imaging atlas of the two most common human colorectal polyps, conventional adenomas and serrated polyps, and their resulting CRC counterparts. Integrative analysis of 128 datasets from 62 participants reveals adenomas arise from WNT-driven expansion of stem cells, while serrated polyps derive from differentiated cells through gastric metaplasia. Metaplasia-associated damage is coupled to a cytotoxic immune microenvironment preceding hypermutation, driven partly by antigen-presentation differences associated with tumor cell-differentiation status. Microsatellite unstable CRCs contain distinct non-metaplastic regions where tumor cells acquire stem cell properties and cytotoxic immune cells are depleted. Our multi-omic atlas provides insights into malignant progression of colorectal polyps and their microenvironment, serving as a framework for precision surveillance and prevention of CRC.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
=======
<li class="list-group-item">Bob Chen, Cherie' R Scurrah, <strong>Eliot T. McKinley</strong>, Alan J Simmons, Marisol A Ramirez-Solano, Xiangzhu Zhu, Nicholas O Markham, Cody N Heiser, Paige N Vega, Andrea Rolong, Hyeyon Kim, Quanhu Sheng, Julia L Drewes, Yuan Zhou, Austin N Southard-Smith, Yanwen Xu, James Ro, Angela L Jones, Frank Revetta, Lynne D Berry, Hiroaki Niitsu, Mirazul Islam, Karin Pelka, Matan Hofree, Jonathan H Chen, Siranush Sarkizova, Kimmie Ng, Marios Giannakis, Genevieve M Boland, Andrew J Aguirre, Ana C Anderson, Orit Rozenblatt-Rosen, Aviv Regev, Nir Hacohen, Kenta Kawasaki, Toshiro Sato, Jeremy A Goettel, William M Grady, Wei Zheng, M Kay Washington, Qiuyin Cai, Cynthia L Sears, James R Goldenring, Jeffrey L Franklin, Timothy Su, Won Jae Huh, Simon Vandekar, Joseph T Roland, Qi Liu, Robert J Coffey, Martha J Shrubsole, and Ken S Lau. (2021) "Differential pre-malignant programs and microenvironment chart distinct paths to malignancy in human colorectal polyps." <em>Cell</em><br><br><details><summary>Abstract</summary><ul>Colorectal cancers (CRCs) arise from precursor polyps whose cellular origins, molecular heterogeneity, and immunogenic potential may reveal diagnostic and therapeutic insights when analyzed at high resolution. We present a single-cell transcriptomic and imaging atlas of the two most common human colorectal polyps, conventional adenomas and serrated polyps, and their resulting CRC counterparts. Integrative analysis of 128 datasets from 62 participants reveals adenomas arise from WNT-driven expansion of stem cells, while serrated polyps derive from differentiated cells through gastric metaplasia. Metaplasia-associated damage is coupled to a cytotoxic immune microenvironment preceding hypermutation, driven partly by antigen-presentation differences associated with tumor cell-differentiation status. Microsatellite unstable CRCs contain distinct non-metaplastic regions where tumor cells acquire stem cell properties and cytotoxic immune cells are depleted. Our multi-omic atlas provides insights into malignant progression of colorectal polyps and their microenvironment, serving as a framework for precision surveillance and prevention of CRC.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
>>>>>>> 9edb1aff55c5ced3256ae9d3470567a2ac9158f1
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://www.biorxiv.org/content/10.1101/2021.01.11.426044v1" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-biorxiv" role="img" aria-label="Preprint"></i>
        Preprint
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://github.com/Ken-Lau-Lab/STAR_Methods" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-github" role="img" aria-label="Github"></i>
        Github
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/34910928/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1016/j.cell.2021.11.031" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li></ul>
<h3 id="published-2020" class="anchored">2020</h3>

<<<<<<< HEAD
<ul class="list-group list-group-flush"><li class="list-group-item">Amrita Banerjee, Charles A. Herring, Bob Chen, Hyeyon Kim, Alan J. Simmons, Austin N. Southard-Smith, Margaret M. Allaman, James R. White, Mary C. Macedonia, <strong>Eliot T. McKinley</strong>, Marisol A. Ramirez-Solano, Elizabeth A. Scoville, Qi Liu, Keith T. Wilson, Robert J. Coffey, M. Kay Washington, Jeremy A. Goettel, and Ken S. Lau. (2020) "Succinate produced by intestinal microbes promotes specification of tuft cells to suppress ileal inflammation." <em>Gastroenterology</em><br><br><details><summary>Abstract</summary><ul>Background &amp; aims: Countries endemic for parasitic infestations have a lower incidence of Crohn's disease (CD) than nonendemic countries, and there have been anecdotal reports of the beneficial effects of helminths in CD patients. Tuft cells in the small intestine sense and direct the immune response against eukaryotic parasites. We investigated the activities of tuft cells in patients with CD and mouse models of intestinal inflammation.<br><br>Methods: We used microscopy to quantify tuft cells in intestinal specimens from patients with ileal CD (n = 19), healthy individuals (n = 14), and TNFΔARE/+ mice, which develop Crohn's-like ileitis. We performed single-cell RNA sequencing, mass spectrometry, and microbiome profiling of intestinal tissues from wild-type and Atoh1-knockout mice, which have expansion of tuft cells, to study interactions between microbes and tuft cell populations. We assessed microbe dependence of tuft cell populations using microbiome depletion, organoids, and microbe transplant experiments. We used multiplex imaging and cytokine assays to assess alterations in inflammatory response following expansion of tuft cells with succinate administration in TNFΔARE/+ and anti-CD3E CD mouse models.<br><br>Results: Inflamed ileal tissues from patients and mice had reduced numbers of tuft cells, compared with healthy individuals or wild-type mice. Expansion of tuft cells was associated with increased expression of genes that regulate the tricarboxylic acid cycle, which resulted from microbe production of the metabolite succinate. Experiments in which we manipulated the intestinal microbiota of mice revealed the existence of an ATOH1-independent population of tuft cells that was sensitive to metabolites produced by microbes. Administration of succinate to mice expanded tuft cells and reduced intestinal inflammation in TNFΔARE/+ mice and anti-CD3E-treated mice, increased GATA3+ cells and type 2 cytokines (IL22, IL25, IL13), and decreased RORGT+ cells and type 17 cytokines (IL23) in a tuft cell-dependent manner.<br><br>Conclusions: We found that tuft cell expansion reduced chronic intestinal inflammation in mice. Strategies to expand tuft cells might be developed for treatment of CD.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
=======
<ul class="list-group list-group-flush"><li class="list-group-item">Amrita Banerjee, Charles A Herring, Bob Chen, Hyeyon Kim, Alan J Simmons, Austin N Southard-Smith, Margaret M Allaman, James R White, Mary C Macedonia, <strong>Eliot T. McKinley</strong>, Marisol A Ramirez-Solano, Elizabeth A Scoville, Qi Liu, Keith T Wilson, Robert J Coffey, M Kay Washington, Jeremy A Goettel, and Ken S Lau. (2020) "Succinate produced by intestinal microbes promotes specification of tuft cells to suppress ileal inflammation." <em>Gastroenterology</em><br><br><details><summary>Abstract</summary><ul>Background &amp; aims: Countries endemic for parasitic infestations have a lower incidence of Crohn's disease (CD) than nonendemic countries, and there have been anecdotal reports of the beneficial effects of helminths in CD patients. Tuft cells in the small intestine sense and direct the immune response against eukaryotic parasites. We investigated the activities of tuft cells in patients with CD and mouse models of intestinal inflammation.<br><br>Methods: We used microscopy to quantify tuft cells in intestinal specimens from patients with ileal CD (n = 19), healthy individuals (n = 14), and TNFΔARE/+ mice, which develop Crohn's-like ileitis. We performed single-cell RNA sequencing, mass spectrometry, and microbiome profiling of intestinal tissues from wild-type and Atoh1-knockout mice, which have expansion of tuft cells, to study interactions between microbes and tuft cell populations. We assessed microbe dependence of tuft cell populations using microbiome depletion, organoids, and microbe transplant experiments. We used multiplex imaging and cytokine assays to assess alterations in inflammatory response following expansion of tuft cells with succinate administration in TNFΔARE/+ and anti-CD3E CD mouse models.<br><br>Results: Inflamed ileal tissues from patients and mice had reduced numbers of tuft cells, compared with healthy individuals or wild-type mice. Expansion of tuft cells was associated with increased expression of genes that regulate the tricarboxylic acid cycle, which resulted from microbe production of the metabolite succinate. Experiments in which we manipulated the intestinal microbiota of mice revealed the existence of an ATOH1-independent population of tuft cells that was sensitive to metabolites produced by microbes. Administration of succinate to mice expanded tuft cells and reduced intestinal inflammation in TNFΔARE/+ mice and anti-CD3E-treated mice, increased GATA3+ cells and type 2 cytokines (IL22, IL25, IL13), and decreased RORGT+ cells and type 17 cytokines (IL23) in a tuft cell-dependent manner.<br><br>Conclusions: We found that tuft cell expansion reduced chronic intestinal inflammation in mice. Strategies to expand tuft cells might be developed for treatment of CD.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
>>>>>>> 9edb1aff55c5ced3256ae9d3470567a2ac9158f1
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/32828819/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1053/j.gastro.2020.08.029" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li>
<li class="list-group-item">Allison S. Cohen, Jun Li, Matthew R. Hight, <strong>Eliot T. McKinley</strong>, Allie Fu, Adria Payne, Yang Liu, Dawei Zhang, Qing Xie, Mingfeng Bai, Gregory D. Ayers, Mohammed Noor Tantawy, Jarrod A. Smith, Frank Revetta, M. Kay Washington, Chanjuan Shi, Nipun Merchant, and H. Charles Manning. (2020) "TSPO-targeted PET and Optical Probes for the Detection and Localization of Premalignant and Malignant Pancreatic Lesions." <em>Clin Cancer Res</em><br><br><details><summary>Abstract</summary><ul>Purpose: Pancreatic cancer is among the most aggressive malignancies and is rarely discovered early. However, pancreatic 'incidentalomas,'' particularly cysts, are frequently identified in asymptomatic patients through anatomic imaging for unrelated causes. Accurate determination of the malignant potential of cystic lesions could lead to life-saving surgery or spare patients with indolent disease undue risk. Current risk assessment of pancreatic cysts requires invasive sampling, with attendant morbidity and sampling errors. Here, we sought to identify imaging biomarkers of high-risk pancreatic cancer precursor lesions.<br><br>Experimental design: Translocator protein (TSPO) expression, which is associated with cholesterol metabolism, was evaluated in premalignant and pancreatic cancer lesions from human and genetically engineered mouse (GEM) tissues. In vivo imaging was performed with [18F]V-1008, a TSPO-targeted PET agent, in two GEM models. For image-guided surgery (IGS), V-1520, a TSPO ligand for near-IR optical imaging based upon the V-1008 pharmacophore, was developed and evaluated.<br><br>Results: TSPO was highly expressed in human and murine pancreatic cancer. Notably, TSPO expression was associated with high-grade, premalignant intraductal papillary mucinous neoplasms (IPMNs) and pancreatic intraepithelial neoplasia (PanIN) lesions. In GEM models, [18F]V-1008 exhibited robust uptake in early pancreatic cancer, detectable by PET. Furthermore, V-1520 localized to premalignant pancreatic lesions and advanced tumors enabling real-time IGS.<br><br>Conclusions: We anticipate that combined TSPO PET/IGS represents a translational approach for precision pancreatic cancer care through discrimination of high-risk indeterminate lesions and actionable surgery.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/32933996/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1158/1078-0432.ccr-20-1214" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li>
<li class="list-group-item">Orit Rozenblatt-Rosen, Aviv Regev, Philipp Oberdoerffer, Tal Nawy, Anna Hupalowska, Jennifer E. Rood, Orr Ashenberg, Ethan Cerami, Robert J. Coffey, Emek Demir, Li Ding, Edward D. Esplin, James M. Ford, Jeremy Goecks, Sharmistha Ghosh, Joe W. Gray, Justin Guinney, Sean E. Hanlon, Shannon K. Hughes, E. Shelley Hwang, Christine A. Iacobuzio-Donahue, Judit Jané-Valbuena, Bruce E. Johnson, Ken S. Lau, Tracy Lively, Sarah A. Mazzilli, Dana Pe’er, Sandro Santagata, Alex K. Shalek, Denis Schapiro, Michael P. Snyder, Peter K. Sorger, Avrum E. Spira, Sudhir Srivastava, Kai Tan, Robert B. West, Elizabeth H. Williams, <strong>Eliot T. McKinley</strong>, and Human Tumor Atlas Network. (2020) "The Human Tumor Atlas Network: charting tumor transitions across space and time at single-cell resolution." <em>Cell</em><br><br><details><summary>Abstract</summary><ul>Crucial transitions in cancer-including tumor initiation, local expansion, metastasis, and therapeutic resistance-involve complex interactions between cells within the dynamic tumor ecosystem. Transformative single-cell genomics technologies and spatial multiplex in situ methods now provide an opportunity to interrogate this complexity at unprecedented resolution. The Human Tumor Atlas Network (HTAN), part of the National Cancer Institute (NCI) Cancer Moonshot Initiative, will establish a clinical, experimental, computational, and organizational framework to generate informative and accessible three-dimensional atlases of cancer transitions for a diverse set of tumor types. This effort complements both ongoing efforts to map healthy organs and previous large-scale cancer genomics approaches focused on bulk sequencing at a single point in time. Generating single-cell, multiparametric, longitudinal atlases and integrating them with clinical outcomes should help identify novel predictive biomarkers and features as well as therapeutically relevant cell types, cell states, and cellular interactions across transitions. The resulting tumor atlases should have a profound impact on our understanding of cancer biology and have the potential to improve cancer detection, prevention, and therapeutic discovery for better precision-medicine treatments of cancer patients and those at risk for cancer.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/32302568/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1016/j.cell.2020.03.053" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li>
<<<<<<< HEAD
<li class="list-group-item">Junzhong Xu, Xiaoyu Jian, Hua Li, Lori R. Arlinghaus, <strong>Eliot T. McKinley</strong>, Sean P. Devan, Benjamin M. Hardy, Jingping Xie, Hakmook Kang, A. Bapsi Chakravarthy, and John C. Gore. (2020) "Magnetic resonance imaging of mean cell size in human breast tumors." <em>Magn Reson Med</em><br><br><details><summary>Abstract</summary><ul>Purpose: Cell size is a fundamental characteristic of all tissues, and changes in cell size in cancer reflect tumor status and response to treatments, such as apoptosis and cell-cycle arrest. Unfortunately, cell size can currently be obtained only by pathological evaluation of tumor tissue samples obtained invasively. Previous imaging approaches are limited to preclinical MRI scanners or require relatively long acquisition times that are impractical for clinical imaging. There is a need to develop cell-size imaging for clinical applications.<br><br>Methods: We propose a clinically feasible IMPULSED (imaging microstructural parameters using limited spectrally edited diffusion) approach that can characterize mean cell sizes in solid tumors. We report the use of a combination of pulse sequences, using different gradient waveforms implemented on clinical MRI scanners and analytical equations based on these waveforms to analyze diffusion-weighted MRI signals and derive specific microstructural parameters such as cell size. We also describe comprehensive validations of this approach using computer simulations, cell experiments in vitro, and animal experiments in vivo and demonstrate applications in preoperative breast cancer patients.<br><br>Results: With fast acquisitions (~7 minutes), IMPULSED can provide high-resolution (1.3 mm in-plane) mapping of mean cell size of human tumors in vivo on clinical 3T MRI scanners. All validations suggest that IMPULSED provides accurate and reliable measurements of mean cell size.<br><br>Conclusion: The proposed IMPULSED method can assess cell-size variations in tumors of breast cancer patients, which may have the potential to assess early response to neoadjuvant therapy.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
=======
<li class="list-group-item">Junzhong Xu, Xiaoyu Jian, Hua L, Lori R. Arlinghau, <strong>Eliot T. McKinley</strong>, Sean P. Devan, Benjamin M. Hardy, Jingping Xie, Hakmook Kang, A. Bapsi Chakravarthy, and John C. Gore. (2020) "Magnetic resonance imaging of mean cell size in human breast tumors." <em>Magn Reson Med</em><br><br><details><summary>Abstract</summary><ul>Purpose: Cell size is a fundamental characteristic of all tissues, and changes in cell size in cancer reflect tumor status and response to treatments, such as apoptosis and cell-cycle arrest. Unfortunately, cell size can currently be obtained only by pathological evaluation of tumor tissue samples obtained invasively. Previous imaging approaches are limited to preclinical MRI scanners or require relatively long acquisition times that are impractical for clinical imaging. There is a need to develop cell-size imaging for clinical applications.<br><br>Methods: We propose a clinically feasible IMPULSED (imaging microstructural parameters using limited spectrally edited diffusion) approach that can characterize mean cell sizes in solid tumors. We report the use of a combination of pulse sequences, using different gradient waveforms implemented on clinical MRI scanners and analytical equations based on these waveforms to analyze diffusion-weighted MRI signals and derive specific microstructural parameters such as cell size. We also describe comprehensive validations of this approach using computer simulations, cell experiments in vitro, and animal experiments in vivo and demonstrate applications in preoperative breast cancer patients.<br><br>Results: With fast acquisitions (~7 minutes), IMPULSED can provide high-resolution (1.3 mm in-plane) mapping of mean cell size of human tumors in vivo on clinical 3T MRI scanners. All validations suggest that IMPULSED provides accurate and reliable measurements of mean cell size.<br><br>Conclusion: The proposed IMPULSED method can assess cell-size variations in tumors of breast cancer patients, which may have the potential to assess early response to neoadjuvant therapy.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
>>>>>>> 9edb1aff55c5ced3256ae9d3470567a2ac9158f1
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/31765494/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1002/mrm.28056" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li>
<<<<<<< HEAD
<li class="list-group-item">Xiaoyu Jiang, Stephanie Dudzinski, Kathryn E. Beckermann, Kirsten Young, <strong>Eliot T. McKinley</strong>, J. Oliver McIntyre, Jeffrey C. Rathmell, Junzhong Xu, and John C. Gore. (2020) "MRI of tumor T cell infiltration in response to checkpoint inhibitor therapy." <em>J Immunother Cancer</em><br><br><details><summary>Abstract</summary><ul>Purpose: Cell size is a fundamental characteristic of all tissues, and changes in cell size in cancer reflect tumor status and response to treatments, such as apoptosis and cell-cycle arrest. Unfortunately, cell size can currently be obtained only by pathological evaluation of tumor tissue samples obtained invasively. Previous imaging approaches are limited to preclinical MRI scanners or require relatively long acquisition times that are impractical for clinical imaging. There is a need to develop cell-size imaging for clinical applications.<br><br>Methods: We propose a clinically feasible IMPULSED (imaging microstructural parameters using limited spectrally edited diffusion) approach that can characterize mean cell sizes in solid tumors. We report the use of a combination of pulse sequences, using different gradient waveforms implemented on clinical MRI scanners and analytical equations based on these waveforms to analyze diffusion-weighted MRI signals and derive specific microstructural parameters such as cell size. We also describe comprehensive validations of this approach using computer simulations, cell experiments in vitro, and animal experiments in vivo and demonstrate applications in preoperative breast cancer patients.<br><br>Results: With fast acquisitions (~7 minutes), IMPULSED can provide high-resolution (1.3 mm in-plane) mapping of mean cell size of human tumors in vivo on clinical 3T MRI scanners. All validations suggest that IMPULSED provides accurate and reliable measurements of mean cell size.<br><br>Conclusion: The proposed IMPULSED method can assess cell-size variations in tumors of breast cancer patients, which may have the potential to assess early response to neoadjuvant therapy.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
=======
<li class="list-group-item">Xiaoyu Jiang, Stephanie Dudzinski, Kathryn E Beckermann, Kirsten Young, <strong>Eliot T. McKinley</strong>, J Oliver McIntyre, Jeffrey C Rathmell, Junzhong Xu, and John C Gore. (2020) "MRI of tumor T cell infiltration in response to checkpoint inhibitor therapy." <em>J Immunother Cancer</em><br><br><details><summary>Abstract</summary><ul>Purpose: Cell size is a fundamental characteristic of all tissues, and changes in cell size in cancer reflect tumor status and response to treatments, such as apoptosis and cell-cycle arrest. Unfortunately, cell size can currently be obtained only by pathological evaluation of tumor tissue samples obtained invasively. Previous imaging approaches are limited to preclinical MRI scanners or require relatively long acquisition times that are impractical for clinical imaging. There is a need to develop cell-size imaging for clinical applications.<br><br>Methods: We propose a clinically feasible IMPULSED (imaging microstructural parameters using limited spectrally edited diffusion) approach that can characterize mean cell sizes in solid tumors. We report the use of a combination of pulse sequences, using different gradient waveforms implemented on clinical MRI scanners and analytical equations based on these waveforms to analyze diffusion-weighted MRI signals and derive specific microstructural parameters such as cell size. We also describe comprehensive validations of this approach using computer simulations, cell experiments in vitro, and animal experiments in vivo and demonstrate applications in preoperative breast cancer patients.<br><br>Results: With fast acquisitions (~7 minutes), IMPULSED can provide high-resolution (1.3 mm in-plane) mapping of mean cell size of human tumors in vivo on clinical 3T MRI scanners. All validations suggest that IMPULSED provides accurate and reliable measurements of mean cell size.<br><br>Conclusion: The proposed IMPULSED method can assess cell-size variations in tumors of breast cancer patients, which may have the potential to assess early response to neoadjuvant therapy.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
>>>>>>> 9edb1aff55c5ced3256ae9d3470567a2ac9158f1
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/32581044/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1136/jitc-2019-000328" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li>
<li class="list-group-item">Won Jae Huh, Hiroaki Niitsu, Brandon Carney, <strong>Eliot T. McKinley</strong>, Jacob L. Houghton, and Robert J. Coffey. (2020) "Identification and Characterization of Unique Neutralizing Antibodies to Mouse EGF Receptor." <em>Gastroenterology</em><br><br><details><summary>Abstract</summary><ul>The epidermal growth factor (EGF) receptor (EGFR) neutralizing monoclonal antibody (mAb) cetuximab is approved by the US Food and Drug Administration for the treatment of advanced colorectal cancer (CRC) and head and neck squamous cell carcinoma, whereas EGFR tyrosine kinase inhibitors are ineffective. Efforts to use EGFR mAb therapy in mouse models of colon cancer have been stymied by the lack of neutralizing mAbs to mouse EGFR. MM-151 is a combination of 3 anti-EGFR mAbs (P1X, P2X, and P3X) shown to have greater receptor down-regulation, immune-effector function, and antiproliferative activity than cetuximab; MM-151 is also able to overcome cetuximab resistance in patients with CRC whose tumors harbor mutations in the extracellular domain (ED) of EGFR, which is divided into ED1–4. Like cetuximab, P1X and P3X bind ED3 of EGFR, whereas P2X binds ED1. Combined P1X/P2X shows synergistic growth inhibition of mouse colonic tumoroids and abrogates EGFR signaling in mouse duodenal tumoroids. Moreover, in vivo, P1X/P2X reverses the histologic features of a mouse model of Ménétrier’s disease, a premalignant disorder for which cetuximab is the first effective medical therapy. Using a recently created Egfr reporter mouse (EgfrEmerald GFP [Em]) that allows direct visualization of endogenous EGFR protein we show that P1X/P2X markedly accelerates internalization and degradation of EGFR in mouse hepatocytes in vitro and in vivo.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/31866246/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1053/j.gastro.2019.12.017" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li>
<li class="list-group-item">Xiaoyu Jiang, <strong>Eliot T. McKinley</strong>, Jingping Xie, Hua Li, Junzhong Xu, and John C. Gore. (2020) "In vivo magnetic resonance imaging of treatment-induced apoptosis." <em>Sci Rep</em><br><br><details><summary>Abstract</summary><ul>Imaging apoptosis could provide an early and specific means to monitor tumor responses to treatment. To date, despite numerous attempts to develop molecular imaging approaches, there is still no widely-accepted and reliable method for in vivo imaging of apoptosis. We hypothesized that the distinct cellular morphologic changes associated with treatment-induced apoptosis, such as cell shrinkage, cytoplasm condensation, and DNA fragmentation, can be detected by temporal diffusion spectroscopy imaging (TDSI). Cetuximab-induced apoptosis was assessed in vitro and in vivo with cetuximab-sensitive (DiFi) and insensitive (HCT-116) human colorectal cancer cell lines by TDSI. TDSI findings were complemented by flow cytometry and immunohistochemistry. Cell cycle analysis and flow cytometry detected apoptotic cell shrinkage in cetuximab-treated DiFi cells, and significant apoptosis was confirmed by histology. TDSI-derived parameters quantified key morphological changes including cell size decreases during apoptosis in responsive tumors that occurred earlier than gross tumor volume regression. TDSI provides a unique measurement of apoptosis by identifying cellular characteristics, particularly cell shrinkage. The method will assist in understanding the underlying biology of solid tumors and predict tumor response to therapies. TDSI is free of any exogenous agent or radiation, and hence is very suitable to be incorporated into clinical applications.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/31266982/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1038/s41598-019-45864-y" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li></ul>
<<<<<<< HEAD
<h3 id="published-2019" class="anchored">2019</h3>
<ul class="list-group list-group-flush"><li class="list-group-item">Qin Zhang, James N. Higginbotham, Dennis K. Jeppesen, Yu-Ping Yang, Wei Li, <strong>Eliot T. McKinley</strong>, Ramona Graves-Deal, Jie Ping, Colleen M. Britain, Kaitlyn A. Dorsett, Celine L. Hartman, David A. Ford, Ryan M. Allen, Kasey C. Vickers, Qi Liu, Jeffrey L. Franklin, and Robert J. Coffey. (2019) "Transfer of Functional Cargo in Exomeres." <em>Cell Rep</em><br><br><details><summary>Abstract</summary><ul>Exomeres are a recently discovered type of extracellular nanoparticle with no known biological function. Herein, we describe a simple ultracentrifugation-based method for separation of exomeres from exosomes. Exomeres are enriched in Argonaute 1-3 and amyloid precursor protein. We identify distinct functions of exomeres mediated by two of their cargo, the β-galactoside α2,6-sialyltransferase 1 (ST6Gal-I) that α2,6- sialylates N-glycans, and the EGFR ligand, amphiregulin (AREG). Functional ST6Gal-I in exomeres can be transferred to cells, resulting in hypersialylation of recipient cell-surface proteins including β1-integrin. AREG-containing exomeres elicit prolonged EGFR and downstream signaling in recipient cells, modulate EGFR trafficking in normal intestinal organoids, and dramatically enhance the growth of colonic tumor organoids. This study provides a simplified method of exomere isolation and demonstrates that exomeres contain and can transfer functional cargo. These findings underscore the heterogeneity of nanoparticles and should accelerate advances in determining the composition and biological functions of exomeres.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/30956133/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1016/j.celrep.2019.01.009" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li></ul>
=======
>>>>>>> 9edb1aff55c5ced3256ae9d3470567a2ac9158f1
<h3 id="published-2018" class="anchored">2018</h3>

<ul class="list-group list-group-flush"><li class="list-group-item">Amrita Banerjee, <strong>Eliot T. McKinley</strong>, Jakob von Moltke, Robert J. Coffey, and Ken S. Lau. (2018) "Interpreting heterogeneity in intestinal tuft cell structure and function." <em>J Clin Invest</em><br><br><details><summary>Abstract</summary><ul>Intestinal tuft cells are a morphologically unique cell type, best characterized by striking microvilli that form an apical tuft. These cells represent approximately 0.5% of gut epithelial cells depending on location. While they are known to express chemosensory receptors, their function has remained unclear. Recently, numerous groups have revealed startling insights into intestinal tuft cell biology. Here, we review the latest developments in understanding this peculiar cell type's structure and function. Recent advances in volumetric microscopy have begun to elucidate tuft cell ultrastructure with respect to its cellular neighbors. Moreover, single-cell approaches have revealed greater diversity in the tuft cell population than previously appreciated and uncovered novel markers to characterize this heterogeneity. Finally, advanced model systems have revealed tuft cells' roles in mucosal healing and orchestrating type 2 immunity against eukaryotic infection. While much remains unknown about intestinal tuft cells, these critical advances have illuminated the physiological importance of these previously understudied cells and provided experimentally tractable tools to interrogate this rare cell population. Tuft cells act as luminal sensors, linking the luminal microbiome to the host immune system, which may make them a potent clinical target for modulating host response to a variety of acute or chronic immune-driven conditions.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/29714721/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1172/jci120330" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li>
<li class="list-group-item">Charles A. Herring, Bob Chen, <strong>Eliot T. McKinley</strong>, and Ken S. Lau. (2018) "Single-Cell Computational Strategies for Lineage Reconstruction in Tissue Systems." <em>Cell Mol Gastroenterol Hepatol</em><br><br><details><summary>Abstract</summary><ul>Function at the organ level manifests itself from a heterogeneous collection of cell types. Cellular heterogeneity emerges from developmental processes by which multipotent progenitor cells make fate decisions and transition to specific cell types through intermediate cell states. Although genetic experimental strategies such as lineage tracing have provided insights into cell lineages, recent developments in single-cell technologies have greatly increased our ability to interrogate distinct cell types, as well as transitional cell states in tissue systems. From single-cell data that describe these intermediate cell states, computational tools have been developed to reconstruct cell-state transition trajectories that model cell developmental processes. These algorithms, although powerful, are still in their infancy, and attention must be paid to their strengths and weaknesses when they are used. Here, we review some of these tools, also referred to as pseudotemporal ordering algorithms, and their associated assumptions and caveats. We hope to provide a rational and generalizable workflow for single-cell trajectory analysis that is intuitive for experimental biologists.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/29713661/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1016/j.jcmgh.2018.01.023" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li>
<<<<<<< HEAD
<li class="list-group-item">Qin Zhang, Dennis K. Jeppesen, James N. Higginbotham, Michelle Demory Beckler, Emily J. Poulin, Alex J. Walsh, Melissa C. Skala, <strong>Eliot T. McKinley</strong>, H. Charles Manning, Matthew R. Hight, Michael L. Schulte, Kimberly R. Watt, Gregory D. Ayers, Melissa M. Wolf, Gabriela Andrejeva, Jeffrey C. Rathmell, Jeffrey L. Franklin, and Robert J. Coffey. (2018) "Mutant KRAS Exosomes Alter the Metabolic State of Recipient Colonic Epithelial Cells." <em>Cell Mol Gastroenterol Hepatol</em><br><br><details><summary>Abstract</summary><ul>In colorectal cancer (CRC) cells, mutant Kirsten rat sarcoma (KRAS) cell-autonomously imparts Warburg-like metabolic changes through induction of Glucose transporter 1 (GLUT-1) (SLC2A1). We previously reported that mutant KRAS has marked effects on the constituents of CRC exosomes, including proteins and enzymes involved in metabolism and glycolysis. The present studies were designed to test whether mutant KRAS exosomes can alter the metabolic state cell-nonautonomously in recipient colonic epithelial cells.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
=======
<li class="list-group-item">Qin Zhang, Dennis K. Jeppesen, James N. Higginbotham, Michelle Demory Beckler, Emily J. Poulin, Alex J. Walsh, Melissa C. Skala, <strong>Eliot T. McKinley</strong>, H. Charles Manning, Matthew R. Hight, Michael L. Schulte, Kimberly R. Watt, G. Daniel Ayers, Melissa M. Wolf, Gabriela Andrejeva, Jeffrey C. Rathmell, Jeffrey L. Franklin, and Robert J. Coffey. (2018) "Mutant KRAS Exosomes Alter the Metabolic State of Recipient Colonic Epithelial Cells." <em>Cell Mol Gastroenterol Hepatol</em><br><br><details><summary>Abstract</summary><ul>In colorectal cancer (CRC) cells, mutant Kirsten rat sarcoma (KRAS) cell-autonomously imparts Warburg-like metabolic changes through induction of Glucose transporter 1 (GLUT-1) (SLC2A1). We previously reported that mutant KRAS has marked effects on the constituents of CRC exosomes, including proteins and enzymes involved in metabolism and glycolysis. The present studies were designed to test whether mutant KRAS exosomes can alter the metabolic state cell-nonautonomously in recipient colonic epithelial cells.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
>>>>>>> 9edb1aff55c5ced3256ae9d3470567a2ac9158f1
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/29930982/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1016%2Fj.jcmgh.2018.01.013" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li>
<li class="list-group-item">Charles A. Herring, Amrita Banerjee, <strong>Eliot T. McKinley</strong>, Alan J. Simmons, Jie Ping, Joseph T. Roland, Jeffrey L. Franklin, Qi Liu, Michael J. Gerdes, Robert J. Coffey, and Ken S. Lau. (2018) "Unsupervised Trajectory Analysis of Single-Cell RNA-Seq and Imaging Data Reveals Alternative Tuft Cell Origins in the Gut." <em>Cell Syst</em><br><br><details><summary>Abstract</summary><ul>Modern single-cell technologies allow multiplexed sampling of cellular states within a tissue. However, computational tools that can infer developmental cell-state transitions reproducibly from such single-cell data are lacking. Here, we introduce p-Creode, an unsupervised algorithm that produces multi-branching graphs from single-cell data, compares graphs with differing topologies, and infers a statistically robust hierarchy of cell-state transitions that define developmental trajectories. We have applied p-Creode to mass cytometry, multiplex immunofluorescence, and single-cell RNA-seq data. As a test case, we validate cell-state-transition trajectories predicted by p-Creode for intestinal tuft cells, a rare, chemosensory cell type. We clarify that tuft cells are specified outside of the Atoh1-dependent secretory lineage in the small intestine. However, p-Creode also predicts, and we confirm, that tuft cells arise from an alternative, Atoh1-driven developmental program in the colon. These studies introduce p-Creode as a reliable method for analyzing large datasets that depict branching transition trajectories.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://github.com/KenLauLab/pCreode" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-github" role="img" aria-label="Github"></i>
        Github
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/29153838/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1016/j.cels.2017.10.012" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li></ul>
<h3 id="published-2017" class="anchored">2017</h3>
<ul class="list-group list-group-flush"><li class="list-group-item"><strong>Eliot T. McKinley</strong>, Yunxia Sui, Yousef Al-Kofahi, Bryan A. Millis, Matthew J. Tyska, Joseph T. Roland, Alberto Santamaria-Pang, Christina L. Ohland, Christian Jobin, Jeffrey L. Franklin, Ken S. Lau, Michael J. Gerdes, and Robert J. Coffey. (2017) "Optimized multiplex immunofluorescence single-cell analysis reveals tuft cell heterogeneity." <em>JCI Insight</em><br><br><details><summary>Abstract</summary><ul>Intestinal tuft cells are a rare, poorly understood cell type recently shown to be a critical mediator of type 2 immune response to helminth infection. Here, we present advances in segmentation algorithms and analytical tools for multiplex immunofluorescence (MxIF), a platform that enables iterative staining of over 60 antibodies on a single tissue section. These refinements have enabled a comprehensive analysis of tuft cell number, distribution, and protein expression profiles as a function of anatomical location and physiological perturbations. Based solely on DCLK1 immunoreactivity, tuft cell numbers were similar throughout the mouse small intestine and colon. However, multiple subsets of tuft cells were uncovered when protein coexpression signatures were examined, including two new intestinal tuft cell markers, Hopx and EGFR phosphotyrosine 1068. Furthermore, we identified dynamic changes in tuft cell number, composition, and protein expression associated with fasting and refeeding and after introduction of microbiota to germ-free mice. These studies provide a foundational framework for future studies of intestinal tuft cell regulation and demonstrate the utility of our improved MxIF computational methods and workflow for understanding cellular heterogeneity in complex tissues in normal and disease states.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://github.com/etmckinley/MxIF" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-github" role="img" aria-label="Github"></i>
        Github
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/28570279/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1172/jci.insight.93487" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li>
<li class="list-group-item">Xiaoyu Jiang, Hua Li, Jingping Xie, <strong>Eliot T. McKinley</strong>, Ping Zhao, John C. Gore, and Junzhong Xu. (2017) "In vivo imaging of cancer cell size and cellularity using temporal diffusion spectroscopy." <em>Magn Reson Med</em><br><br><details><summary>Abstract</summary><ul>Purpose: A temporal diffusion MRI spectroscopy based approach has been developed to quantify cancer cell size and density in vivo.<br><br>Methods: A novel imaging microstructural parameters using limited spectrally edited diffusion (IMPULSED) method selects a specific limited diffusion spectral window for an accurate quantification of cell sizes ranging from 10 to 20 μm in common solid tumors. In practice, it is achieved by a combination of a single long diffusion time pulsed gradient spin echo (PGSE) and three low-frequency oscillating gradient spin echo (OGSE) acquisitions. To validate our approach, hematoxylin and eosin staining and immunostaining of cell membranes, in concert with whole slide imaging, were used to visualize nuclei and cell boundaries, and hence, enabled accurate estimates of cell size and cellularity.<br><br>Results: Based on a two compartment model (incorporating intra- and extracellular spaces), accurate estimates of cell sizes were obtained in vivo for three types of human colon cancers. The IMPULSED-derived apparent cellularities showed a stronger correlation (r = 0.81; P &lt; 0.0001) with histology-derived cellularities than conventional ADCs (r = -0.69; P &lt; 0.03).<br><br>Conclusion: The IMPULSED approach samples a specific region of temporal diffusion spectra with enhanced sensitivity to length scales of 10-20 μm, and enables measurements of cell sizes and cellularities in solid tumors in vivo. Magn Reson Med 78:156-164, 2017. © 2016 International Society for Magnetic Resonance in Medicine.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/27495144/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1002/mrm.26356" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li></ul>
<h3 id="published-2016" class="anchored">2016</h3>
<<<<<<< HEAD
<ul class="list-group list-group-flush"><li class="list-group-item">Alan J. Simmons, Cherie' R. Scurrah, <strong>Eliot T. McKinley</strong>, Charles A. Herring, Jonathan M. Irish, M. Kay Washington, Robert J. Coffey, and Ken S. Lau. (2016) "Impaired coordination between signaling pathways is revealed in human colorectal cancer using single-cell mass cytometry of archival tissue blocks." <em>Sci Signal</em><br><br><details><summary>Abstract</summary><ul>Cellular heterogeneity poses a substantial challenge to understanding tissue-level phenotypes and confounds conventional bulk analyses. To analyze signaling at the single-cell level in human tissues, we applied mass cytometry using cytometry time of flight to formalin-fixed, paraffin-embedded (FFPE) normal and diseased intestinal specimens. This technique, called FFPE-DISSECT (disaggregation for intracellular signaling in single epithelial cells from tissue), is a single-cell approach to characterizing signaling states in embedded tissue samples. We applied FFPE-DISSECT coupled to mass cytometry and found differential signaling by tumor necrosis factor-α in intestinal enterocytes, goblet cells, and enteroendocrine cells, implicating the downstream RAS-RAF-MEK pathway in determining goblet cell identity. Application of this technique and computational analyses to human colon specimens confirmed the reduced differentiation in colorectal cancer (CRC) compared to normal colon and revealed increased intratissue and intertissue heterogeneity in CRC with quantitative changes in the regulation of signaling pathways. Specifically, coregulation of the kinases p38 and ERK, the translation regulator 4EBP1, and the transcription factor CREB in proliferating normal colon cells was lost in CRC. Our data suggest that this single-cell approach, applied in conjunction with genomic annotation, enables the rapid and detailed characterization of cellular heterogeneity from clinical repositories of embedded human tissues. This technique can be used to derive cellular landscapes from archived patient samples (beyond CRC) and as a high-resolution tool for disease characterization and subtyping.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
=======
<ul class="list-group list-group-flush"><li class="list-group-item">Alan J Simmons, Cherie' R Scurrah, <strong>Eliot T. McKinley</strong>, Charles A Herring, Jonathan M Irish, M Kay Washington, Robert J Coffey, and Ken S Lau. (2016) "Impaired coordination between signaling pathways is revealed in human colorectal cancer using single-cell mass cytometry of archival tissue blocks." <em>Sci Signal</em><br><br><details><summary>Abstract</summary><ul>Cellular heterogeneity poses a substantial challenge to understanding tissue-level phenotypes and confounds conventional bulk analyses. To analyze signaling at the single-cell level in human tissues, we applied mass cytometry using cytometry time of flight to formalin-fixed, paraffin-embedded (FFPE) normal and diseased intestinal specimens. This technique, called FFPE-DISSECT (disaggregation for intracellular signaling in single epithelial cells from tissue), is a single-cell approach to characterizing signaling states in embedded tissue samples. We applied FFPE-DISSECT coupled to mass cytometry and found differential signaling by tumor necrosis factor-α in intestinal enterocytes, goblet cells, and enteroendocrine cells, implicating the downstream RAS-RAF-MEK pathway in determining goblet cell identity. Application of this technique and computational analyses to human colon specimens confirmed the reduced differentiation in colorectal cancer (CRC) compared to normal colon and revealed increased intratissue and intertissue heterogeneity in CRC with quantitative changes in the regulation of signaling pathways. Specifically, coregulation of the kinases p38 and ERK, the translation regulator 4EBP1, and the transcription factor CREB in proliferating normal colon cells was lost in CRC. Our data suggest that this single-cell approach, applied in conjunction with genomic annotation, enables the rapid and detailed characterization of cellular heterogeneity from clinical repositories of embedded human tissues. This technique can be used to derive cellular landscapes from archived patient samples (beyond CRC) and as a high-resolution tool for disease characterization and subtyping.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
>>>>>>> 9edb1aff55c5ced3256ae9d3470567a2ac9158f1
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/27729552/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1126/scisignal.aah4413" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li></ul>
<h3 id="published-2015" class="anchored">2015</h3>

<<<<<<< HEAD
<ul class="list-group list-group-flush"><li class="list-group-item">Alan J. Simmons, Amrita Banerjee, <strong>Eliot T. McKinley</strong>, Cherie' R. Scurrah, Charles A. Herring, Leslie S. Gewin, Ryota Masuzaki, Seth J. Karp, Jeffrey L. Franklin, Michael J. Gerdes, Jonathan M. Irish, Robert J. Coffey, and Ken S. Lau. (2015) "Cytometry-based single-cell analysis of intact epithelial signaling reveals MAPK activation divergent from TNF-α-induced apoptosis in vivo." <em>Mol Syst Biol</em><br><br><details><summary>Abstract</summary><ul>Understanding heterogeneous cellular behaviors in a complex tissue requires the evaluation of signaling networks at single-cell resolution. However, probing signaling in epithelial tissues using cytometry-based single-cell analysis has been confounded by the necessity of single-cell dissociation, where disrupting cell-to-cell connections inherently perturbs native cell signaling states. Here, we demonstrate a novel strategy (Disaggregation for Intracellular Signaling in Single Epithelial Cells from Tissue-DISSECT) that preserves native signaling for Cytometry Time-of-Flight (CyTOF) and fluorescent flow cytometry applications. A 21-plex CyTOF analysis encompassing core signaling and cell-identity markers was performed on the small intestinal epithelium after systemic tumor necrosis factor-alpha (TNF-α) stimulation. Unsupervised and supervised analyses robustly selected signaling features that identify a unique subset of epithelial cells that are sensitized to TNF-α-induced apoptosis in the seemingly homogeneous enterocyte population. Specifically, p-ERK and apoptosis are divergently regulated in neighboring enterocytes within the epithelium, suggesting a mechanism of contact-dependent survival. Our novel single-cell approach can broadly be applied, using both CyTOF and multi-parameter flow cytometry, for investigating normal and diseased cell states in a wide range of epithelial tissues.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
=======
<ul class="list-group list-group-flush"><li class="list-group-item">Alan J Simmons, Amrita Banerjee, <strong>Eliot T. McKinley</strong>, Cherie' R Scurrah, Charles A Herring, Leslie S Gewin, Ryota Masuzaki, Seth J Karp, Jeffrey L Franklin, Michael J Gerdes, Jonathan M Irish, Robert J Coffey, and Ken S Lau. (2015) "Cytometry-based single-cell analysis of intact epithelial signaling reveals MAPK activation divergent from TNF-α-induced apoptosis in vivo." <em>Mol Syst Biol</em><br><br><details><summary>Abstract</summary><ul>Understanding heterogeneous cellular behaviors in a complex tissue requires the evaluation of signaling networks at single-cell resolution. However, probing signaling in epithelial tissues using cytometry-based single-cell analysis has been confounded by the necessity of single-cell dissociation, where disrupting cell-to-cell connections inherently perturbs native cell signaling states. Here, we demonstrate a novel strategy (Disaggregation for Intracellular Signaling in Single Epithelial Cells from Tissue-DISSECT) that preserves native signaling for Cytometry Time-of-Flight (CyTOF) and fluorescent flow cytometry applications. A 21-plex CyTOF analysis encompassing core signaling and cell-identity markers was performed on the small intestinal epithelium after systemic tumor necrosis factor-alpha (TNF-α) stimulation. Unsupervised and supervised analyses robustly selected signaling features that identify a unique subset of epithelial cells that are sensitized to TNF-α-induced apoptosis in the seemingly homogeneous enterocyte population. Specifically, p-ERK and apoptosis are divergently regulated in neighboring enterocytes within the epithelium, suggesting a mechanism of contact-dependent survival. Our novel single-cell approach can broadly be applied, using both CyTOF and multi-parameter flow cytometry, for investigating normal and diseased cell states in a wide range of epithelial tissues.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
>>>>>>> 9edb1aff55c5ced3256ae9d3470567a2ac9158f1
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/26519361/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.15252/msb.20156282" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li>
<li class="list-group-item">Jason R. Buck, <strong>Eliot T. McKinley</strong>, Allie Fu, Ty W. Abel, Reid C. Thompson, Lola Chambless, Jennifer M. Watchmaker, James P. Harty, Michael K. Cooper, and H. Charles Manning. (2015) "Preclinical TSPO Ligand PET to Visualize Human Glioma Xenotransplants: A Preliminary Study." <em>PLoS One</em><br><br><details><summary>Abstract</summary><ul>Current positron emission tomography (PET) imaging biomarkers for detection of infiltrating gliomas are limited. Translocator protein (TSPO) is a novel and promising biomarker for glioma PET imaging. To validate TSPO as a potential target for molecular imaging of glioma, TSPO expression was assayed in a tumor microarray containing 37 high-grade (III, IV) gliomas. TSPO staining was detected in all tumor specimens. Subsequently, PET imaging was performed with an aryloxyanilide-based TSPO ligand, [18F]PBR06, in primary orthotopic xenograft models of WHO grade III and IV gliomas. Selective uptake of [18F]PBR06 in engrafted tumor was measured. Furthermore, PET imaging with [18F]PBR06 demonstrated infiltrative glioma growth that was undetectable by traditional magnetic resonance imaging (MRI). Preliminary PET with [18F]PBR06 demonstrated a preferential tumor-to-normal background ratio in comparison to 2-deoxy-2-[18F]fluoro-D-glucose ([18F]FDG). These results suggest that TSPO PET imaging with such high-affinity radiotracers may represent a novel strategy to characterize distinct molecular features of glioma growth, as well as better define the extent of glioma infiltration for therapeutic purposes.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/26517124/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1371%2Fjournal.pone.0141659" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li>
<li class="list-group-item"><strong>Eliot T. McKinley</strong>, Jennifer M. Watchmaker, A. Bapsi Chakravarthy, Jeffrey A. Meyerhardt, Jeffrey A. Engelman, Ronald C. Walker, M. Kay Washington, Robert J. Coffey, and H. Charles Manning. (2015) "[18F]-FLT PET to predict early response to neoadjuvant therapy in KRAS wild-type rectal cancer: a pilot study." <em>Ann Nucl Med</em><br><br><details><summary>Abstract</summary><ul>Object: This pilot study evaluated the utility of 3'-deoxy-3'[18F]-fluorothymidine ([(18)F]-FLT) positron emission tomography (PET) to predict response to neoadjuvant therapy that included cetuximab in patients with wild-type KRAS rectal cancers.<br><br>Methods: Baseline [(18)F]-FLT PET was collected prior to treatment initiation. Follow-up [(18)F]-FLT was collected after three weekly infusions of cetuximab, and following a combined regimen of cetuximab, 5-FU, and radiation. Imaging-matched biopsies were collected with each PET study.<br><br>Results: Diminished [(18)F]-FLT PET was observed in 3/4 of patients following cetuximab treatment alone and in all patients following combination therapy. Reduced [(18)F]-FLT PET following combination therapy predicted disease-free status at surgery. Overall, [(18)F]-FLT PET agreed with Ki67 immunoreactivity from biopsy samples and surgically resected tissue, and was predictive of treatment-induced rise in p27 levels.<br><br>Conclusion: These results suggest that [(18)F]-FLT PET is a promising imaging biomarker to predict response to neoadjuvant therapy that included EGFR blockade with cetuximab in patients with rectal cancer.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/25899481/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1007%2Fs12149-015-0974-6" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li>
<<<<<<< HEAD
<li class="list-group-item">Yiu-Yin Cheung, Michael L. Nickels, <strong>Eliot T. McKinley</strong>, Jason R. Buck, and H. Charles Manning. (2015) "High-yielding, automated production of 3′-deoxy-3′-[18F]fluorothymidine using a modified Bioscan Coincidence FDG reaction module." <em>Appl Radiat Isot</em><br><br><details><summary>Abstract</summary><ul>Introduction: High-yielding, automated production of a PET tracer that reflects proliferation, 3'-deoxy-3'-[(18)F]fluorothymidine ([(18)F]FLT), is reported using a modified Bioscan Coincidence FDG reaction module.<br><br>Methods: Production of [(18)F]FLT was implemented through: (1) modification of an original FDG manifold; (2) application of an alternate time sequence; and (3) altered solid-phase extraction (SPE) purification. Quality control testing, including standard radiochemical figures of merit and preclinical positron emission tomography (PET) imaging, was carried out.<br><br>Results: High decay-corrected yields of [(18)F]FLT (16-39%) were reproducibly obtained. The product exhibited very high specific activity (4586.9TBq/mmol; 123,969Ci/mmol) and radiochemical purity (&gt;99%). Overall, the [(18)F]FLT produced in this manner was superior to typical productions that utilized a GE TRACERlab FXF-N reaction module. Additionally, purification with SPE cartridges, followed by manual elution, accelerated overall run time and resulted in a two-fold increase in [(18)F]FLT concentration. PET imaging showed the [(18)F]FLT produced by this method was highly suitable for non-invasive tumor imaging in mice.<br><br>Conclusions: The Bioscan Coincidence GE FDG Reaction Module was readily adapted to reproducibly provide [(18)F]FLT in high yield, specific activity, and radiochemical purity. The approach was suitable to provide sufficient amounts of material for preclinical studies.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
=======
<li class="list-group-item">Yiu-YinCheung, Michael L.Nickels, <strong>Eliot T. McKinley</strong>, Jason R.Buck, and H. CharlesManning. (2015) "High-yielding, automated production of 3′-deoxy-3′-[18F]fluorothymidine using a modified Bioscan Coincidence FDG reaction module." <em>Appl Radiat Isot</em><br><br><details><summary>Abstract</summary><ul>Introduction: High-yielding, automated production of a PET tracer that reflects proliferation, 3'-deoxy-3'-[(18)F]fluorothymidine ([(18)F]FLT), is reported using a modified Bioscan Coincidence FDG reaction module.<br><br>Methods: Production of [(18)F]FLT was implemented through: (1) modification of an original FDG manifold; (2) application of an alternate time sequence; and (3) altered solid-phase extraction (SPE) purification. Quality control testing, including standard radiochemical figures of merit and preclinical positron emission tomography (PET) imaging, was carried out.<br><br>Results: High decay-corrected yields of [(18)F]FLT (16-39%) were reproducibly obtained. The product exhibited very high specific activity (4586.9TBq/mmol; 123,969Ci/mmol) and radiochemical purity (&gt;99%). Overall, the [(18)F]FLT produced in this manner was superior to typical productions that utilized a GE TRACERlab FXF-N reaction module. Additionally, purification with SPE cartridges, followed by manual elution, accelerated overall run time and resulted in a two-fold increase in [(18)F]FLT concentration. PET imaging showed the [(18)F]FLT produced by this method was highly suitable for non-invasive tumor imaging in mice.<br><br>Conclusions: The Bioscan Coincidence GE FDG Reaction Module was readily adapted to reproducibly provide [(18)F]FLT in high yield, specific activity, and radiochemical purity. The approach was suitable to provide sufficient amounts of material for preclinical studies.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
>>>>>>> 9edb1aff55c5ced3256ae9d3470567a2ac9158f1
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/25531913/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1016%2Fj.apradiso.2014.11.011" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li></ul>
<h3 id="published-2014" class="anchored">2014</h3>

<<<<<<< HEAD
<ul class="list-group list-group-flush"><li class="list-group-item"><strong>Eliot T. McKinley</strong>, Ping Zhao, Robert J. Coffey, M. Kay Washington, and H. Charles Manning. (2014) "3'-Deoxy-3'-[18F]-Fluorothymidine PET imaging reflects PI3K-mTOR-mediated pro-survival response to targeted therapy in colorectal cancer." <em>PLoS One</em><br><br><details><summary>Abstract</summary><ul>Biomarkers that predict response to targeted therapy in oncology are an essential component of personalized medicine. In preclinical treatment response studies that featured models of wild-type KRAS or mutant BRAF colorectal cancer treated with either cetuximab or vemurafenib, respectively, we illustrate that [(18)F]-FLT PET, a non-invasive molecular imaging readout of thymidine salvage, closely reflects pro-survival responses to targeted therapy that are mediated by PI3K-mTOR activity. Activation of pro-survival mechanisms forms the basis of numerous modes of resistance. Therefore, we conclude that [(18)F]-FLT PET may serve a novel and potentially critical role to predict tumors that exhibit molecular features that tend to reflect recalcitrance to MAPK-targeted therapy. Though these studies focused on colorectal cancer, we envision that the results may be applicable to other solid tumors as well.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
=======
<ul class="list-group list-group-flush"><li class="list-group-item"><strong>Eliot T. McKinley</strong>, Ping Zhao, Robert J Coffey, M. Kay Washington, and H. Charles Manning. (2014) "3'-Deoxy-3'-[18F]-Fluorothymidine PET imaging reflects PI3K-mTOR-mediated pro-survival response to targeted therapy in colorectal cancer." <em>PLoS One</em><br><br><details><summary>Abstract</summary><ul>Biomarkers that predict response to targeted therapy in oncology are an essential component of personalized medicine. In preclinical treatment response studies that featured models of wild-type KRAS or mutant BRAF colorectal cancer treated with either cetuximab or vemurafenib, respectively, we illustrate that [(18)F]-FLT PET, a non-invasive molecular imaging readout of thymidine salvage, closely reflects pro-survival responses to targeted therapy that are mediated by PI3K-mTOR activity. Activation of pro-survival mechanisms forms the basis of numerous modes of resistance. Therefore, we conclude that [(18)F]-FLT PET may serve a novel and potentially critical role to predict tumors that exhibit molecular features that tend to reflect recalcitrance to MAPK-targeted therapy. Though these studies focused on colorectal cancer, we envision that the results may be applicable to other solid tumors as well.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
>>>>>>> 9edb1aff55c5ced3256ae9d3470567a2ac9158f1
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/25247710/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1371/journal.pone.0108193" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li>
<<<<<<< HEAD
<li class="list-group-item">Anne E. Powell, Gregory Vlacich, Zhen-Yang Zhao, <strong>Eliot T. McKinley</strong>, M. Kay Washington, H. Charles Manning, and Robert J. Coffey. (2014) "Inducible loss of one Apc allele in Lrig1-expressing progenitor cells results in multiple distal colonic tumors with features of familial adenomatous polyposis." <em>Am J Physiol Gastrointest Liver Physiol</em><br><br><details><summary>Abstract</summary><ul>Individuals with familial adenomatous polyposis (FAP) harbor a germline mutation in adenomatous polyposis coli (APC). The major clinical manifestation is development of multiple colonic tumors at a young age due to stochastic loss of the remaining APC allele. Extracolonic features, including periampullary tumors, gastric abnormalities, and congenital hypertrophy of the retinal pigment epithelium, may occur. The objective of this study was to develop a mouse model that simulates these features of FAP. We combined our Lrig1-CreERT2/+ mice with Apcfl/+ mice, eliminated one copy of Apc in leucine-rich repeats and immunoglobulin-like domains protein 1 (Lrig1)-positive (Lrig1(+)) progenitor cells with tamoxifen injection, and monitored tumor formation in the colon by colonoscopy and PET. Initial loss of one Apc allele in Lrig1(+) cells results in a predictable pattern of preneoplastic changes, culminating in multiple distal colonic tumors within 50 days of induction, as well as the extracolonic manifestations of FAP mentioned above. We show that tumor formation can be monitored by noninvasive PET imaging. This inducible stem cell-driven model recapitulates features of FAP and offers a tractable platform on which therapeutic interventions can be monitored over time by colonoscopy and noninvasive imaging.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
=======
<li class="list-group-item">Anne E Powell, Gregory Vlacich, Zhen-Yang Zhao, <strong>Eliot T. McKinley</strong>, M Kay Washington, H Charles Manning, and Robert J Coffey. (2014) "Inducible loss of one Apc allele in Lrig1-expressing progenitor cells results in multiple distal colonic tumors with features of familial adenomatous polyposis." <em>Am J Physiol Gastrointest Liver Physiol</em><br><br><details><summary>Abstract</summary><ul>Individuals with familial adenomatous polyposis (FAP) harbor a germline mutation in adenomatous polyposis coli (APC). The major clinical manifestation is development of multiple colonic tumors at a young age due to stochastic loss of the remaining APC allele. Extracolonic features, including periampullary tumors, gastric abnormalities, and congenital hypertrophy of the retinal pigment epithelium, may occur. The objective of this study was to develop a mouse model that simulates these features of FAP. We combined our Lrig1-CreERT2/+ mice with Apcfl/+ mice, eliminated one copy of Apc in leucine-rich repeats and immunoglobulin-like domains protein 1 (Lrig1)-positive (Lrig1(+)) progenitor cells with tamoxifen injection, and monitored tumor formation in the colon by colonoscopy and PET. Initial loss of one Apc allele in Lrig1(+) cells results in a predictable pattern of preneoplastic changes, culminating in multiple distal colonic tumors within 50 days of induction, as well as the extracolonic manifestations of FAP mentioned above. We show that tumor formation can be monitored by noninvasive PET imaging. This inducible stem cell-driven model recapitulates features of FAP and offers a tractable platform on which therapeutic interventions can be monitored over time by colonoscopy and noninvasive imaging.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
>>>>>>> 9edb1aff55c5ced3256ae9d3470567a2ac9158f1
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/24833705/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1152/ajpgi.00358.2013" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li></ul>
<h3 id="published-2013" class="anchored">2013</h3>
<<<<<<< HEAD
<ul class="list-group list-group-flush"><li class="list-group-item"><strong>Eliot T. McKinley</strong>, Huiling Liu, W. Hayes McDonald, Weifeng Luo, Ping Zhao, Robert J. Coffey, Steven K. Hanks, and H. Charles Manning. (2013) "Global Phosphotyrosine Proteomics Identifies PKCδ as a Marker of Responsiveness to Src Inhibition in Colorectal Cancer." <em>PLoS One</em><br><br><details><summary>Abstract</summary><ul>Sensitive and specific biomarkers of protein kinase inhibition can be leveraged to accelerate drug development studies in oncology by associating early molecular responses with target inhibition. In this study, we utilized unbiased shotgun phosphotyrosine (pY) proteomics to discover novel biomarkers of response to dasatinib, a small molecule Src-selective inhibitor, in preclinical models of colorectal cancer (CRC). We performed unbiased mass spectrometry shotgun pY proteomics to reveal the pY proteome of cultured HCT-116 colonic carcinoma cells, and then extended this analysis to HCT-116 xenograft tumors to identify pY biomarkers of dasatinib-responsiveness in vivo. Major dasatinib-responsive pY sites in xenograft tumors included sites on delta-type protein kinase C (PKCδ), CUB-domain-containing protein 1 (CDCP1), Type-II SH2-domain-containing inositol 5-phosphatase (SHIP2), and receptor protein-tyrosine phosphatase alpha (RPTPα). The pY313 site PKCδ was further supported as a relevant biomarker of dasatinib-mediated Src inhibition in HCT-116 xenografts by immunohistochemistry and immunoblotting with a phosphospecific antibody. Reduction of PKCδ pY313 was further correlated with dasatinib-mediated inhibition of Src and diminished growth as spheroids of a panel of human CRC cell lines. These studies reveal PKCδ pY313 as a promising readout of Src inhibition in CRC and potentially other solid tumors and may reflect responsiveness to dasatinib in a subset of colorectal cancers.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
=======
<ul class="list-group list-group-flush"><li class="list-group-item"><strong>Eliot T. McKinley</strong>, Huiling Liu, W Hayes McDonald, Weifeng Luo, Ping Zhao, Robert J Coffey, Steven K Hanks, and H Charles Manning. (2013) "Global Phosphotyrosine Proteomics Identifies PKCδ as a Marker of Responsiveness to Src Inhibition in Colorectal Cancer." <em>PLoS One</em><br><br><details><summary>Abstract</summary><ul>Sensitive and specific biomarkers of protein kinase inhibition can be leveraged to accelerate drug development studies in oncology by associating early molecular responses with target inhibition. In this study, we utilized unbiased shotgun phosphotyrosine (pY) proteomics to discover novel biomarkers of response to dasatinib, a small molecule Src-selective inhibitor, in preclinical models of colorectal cancer (CRC). We performed unbiased mass spectrometry shotgun pY proteomics to reveal the pY proteome of cultured HCT-116 colonic carcinoma cells, and then extended this analysis to HCT-116 xenograft tumors to identify pY biomarkers of dasatinib-responsiveness in vivo. Major dasatinib-responsive pY sites in xenograft tumors included sites on delta-type protein kinase C (PKCδ), CUB-domain-containing protein 1 (CDCP1), Type-II SH2-domain-containing inositol 5-phosphatase (SHIP2), and receptor protein-tyrosine phosphatase alpha (RPTPα). The pY313 site PKCδ was further supported as a relevant biomarker of dasatinib-mediated Src inhibition in HCT-116 xenografts by immunohistochemistry and immunoblotting with a phosphospecific antibody. Reduction of PKCδ pY313 was further correlated with dasatinib-mediated inhibition of Src and diminished growth as spheroids of a panel of human CRC cell lines. These studies reveal PKCδ pY313 as a promising readout of Src inhibition in CRC and potentially other solid tumors and may reflect responsiveness to dasatinib in a subset of colorectal cancers.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
>>>>>>> 9edb1aff55c5ced3256ae9d3470567a2ac9158f1
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/24260357/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1371/journal.pone.0080207" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li>
<li class="list-group-item">Dewei Tang, <strong>Eliot T. McKinley</strong>, Matthew R. Hight, Md. Imam Uddin, Joel M. Harp, Allie Fu, Michael L. Nickels, Jason R. Buck, and H. Charles Manning. (2013) "Synthesis and Structure-Activity Relationships of 5,6,7-substituted Pyrazolopyrimidines: Discovery of a novel TSPO PET Ligand for Cancer Imaging." <em>J Med Chem</em><br><br><details><summary>Abstract</summary><ul>Focused library synthesis and structure-activity relationship development of 5,6,7-substituted pyrazolopyrimidines led to the discovery of 2-(5,7-diethyl-2-(4-(2-fluoroethoxy)phenyl)pyrazolo[1,5-a]pyrimidin-3-yl)-N,N-diethylacetamide (6b), a novel translocator protein (TSPO) ligand exhibiting a 36-fold enhancement in affinity compared to another pyrazolopyrimidine-based TSPO ligand, 6a (DPA-714). Radiolabeling with fluorine-18 ((18)F) facilitated production of 2-(5,7-diethyl-2-(4-(2-[(18)F]fluoroethoxy)phenyl)pyrazolo[1,5-a]pyrimidin-3-yl)-N,N-diethylacetamide ((18)F-6b) in high radiochemical yield and specific activity. In vivo studies of (18)F-6b were performed which illuminated this agent as an improved probe for molecular imaging of TSPO-expressing cancers.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/23521048/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1021%2Fjm4001874" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li>
<<<<<<< HEAD
<li class="list-group-item"><strong>Eliot T. McKinley</strong>, Gregory D. Ayers, R. Adam Smith, Samir A. Saleh, Ping Zhao, M. Kay Washington, Robert J. Coffey, and H. Charles Manning. (2013) "Limits of [18F]-FLT PET as a biomarker of proliferation in oncology." <em>PLoS One</em><br><br><details><summary>Abstract</summary><ul>Background: Non-invasive imaging biomarkers of cellular proliferation hold great promise for quantifying response to personalized medicine in oncology. An emerging approach to assess tumor proliferation utilizes the positron emission tomography (PET) tracer 3'-deoxy-3'[(18)F]-fluorothymidine, [(18)F]-FLT. Though several studies have associated serial changes in [(18)F]-FLT-PET with elements of therapeutic response, the degree to which [(18)F]-FLT-PET quantitatively reflects proliferative index has been continuously debated for more that a decade. The goal of this study was to elucidate quantitative relationships between [(18)F]-FLT-PET and cellular metrics of proliferation in treatment naïve human cell line xenografts commonly employed in cancer research.<br><br>Methods and findings: [(18)F]-FLT-PET was conducted in human cancer xenograft-bearing mice. Quantitative relationships between PET, thymidine kinase 1 (TK1) protein levels and immunostaining for proliferation markers (Ki67, TK1, PCNA) were evaluated using imaging-matched tumor specimens. Overall, we determined that [(18)F]-FLT-PET reflects TK1 protein levels, yet the cell cycle specificity of TK1 expression and the extent to which tumors utilize thymidine salvage for DNA synthesis decouple [(18)F]-FLT-PET data from standard estimates of proliferative index.<br><br>Conclusions: Our findings illustrate that [(18)F]-FLT-PET reflects tumor proliferation as a function of thymidine salvage pathway utilization. Unlike more general proliferation markers, such as Ki67, [(18)F]-FLT PET reflects proliferative indices to variable and potentially unreliable extents. [(18)F]-FLT-PET cannot discriminate moderately proliferative, thymidine salvage-driven tumors from those of high proliferative index that rely primarily upon de novo thymidine synthesis. Accordingly, the magnitude of [(18)F]-FLT uptake should not be considered a surrogate of proliferative index. These data rationalize the diversity of [(18)F]-FLT-PET correlative results previously reported and suggest future best-practices when [(18)F]-FLT-PET is employed in oncology.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
=======
<li class="list-group-item"><strong>Eliot T. McKinley</strong>, Gregory D Ayers, R Adam Smith, Samir A Saleh, Ping Zhao, Mary Kay Washington, Robert J Coffey, and H Charles Manning. (2013) "Limits of [18F]-FLT PET as a biomarker of proliferation in oncology." <em>PLoS One</em><br><br><details><summary>Abstract</summary><ul>Background: Non-invasive imaging biomarkers of cellular proliferation hold great promise for quantifying response to personalized medicine in oncology. An emerging approach to assess tumor proliferation utilizes the positron emission tomography (PET) tracer 3'-deoxy-3'[(18)F]-fluorothymidine, [(18)F]-FLT. Though several studies have associated serial changes in [(18)F]-FLT-PET with elements of therapeutic response, the degree to which [(18)F]-FLT-PET quantitatively reflects proliferative index has been continuously debated for more that a decade. The goal of this study was to elucidate quantitative relationships between [(18)F]-FLT-PET and cellular metrics of proliferation in treatment naïve human cell line xenografts commonly employed in cancer research.<br><br>Methods and findings: [(18)F]-FLT-PET was conducted in human cancer xenograft-bearing mice. Quantitative relationships between PET, thymidine kinase 1 (TK1) protein levels and immunostaining for proliferation markers (Ki67, TK1, PCNA) were evaluated using imaging-matched tumor specimens. Overall, we determined that [(18)F]-FLT-PET reflects TK1 protein levels, yet the cell cycle specificity of TK1 expression and the extent to which tumors utilize thymidine salvage for DNA synthesis decouple [(18)F]-FLT-PET data from standard estimates of proliferative index.<br><br>Conclusions: Our findings illustrate that [(18)F]-FLT-PET reflects tumor proliferation as a function of thymidine salvage pathway utilization. Unlike more general proliferation markers, such as Ki67, [(18)F]-FLT PET reflects proliferative indices to variable and potentially unreliable extents. [(18)F]-FLT-PET cannot discriminate moderately proliferative, thymidine salvage-driven tumors from those of high proliferative index that rely primarily upon de novo thymidine synthesis. Accordingly, the magnitude of [(18)F]-FLT uptake should not be considered a surrogate of proliferative index. These data rationalize the diversity of [(18)F]-FLT-PET correlative results previously reported and suggest future best-practices when [(18)F]-FLT-PET is employed in oncology.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
>>>>>>> 9edb1aff55c5ced3256ae9d3470567a2ac9158f1
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/23554961/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1371/journal.pone.0058938" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li>
<<<<<<< HEAD
<li class="list-group-item"><strong>Eliot T. McKinley</strong>, R. Adam Smith, Ping Zhao, Allie Fu, Samir A. Saleh, Md. Imam Uddin, M. Kay Washington, Robert J. Coffey, and H. Charles Manning. (2013) "3'-Deoxy-3'-18F-fluorothymidine PET predicts response to (V600E)BRAF-targeted therapy in preclinical models of colorectal cancer." <em>J Nucl Med</em><br><br><details><summary>Abstract</summary><ul>Selective inhibition of oncogenic targets and associated signaling pathways forms the basis of personalized cancer medicine. The clinical success of (V600E)BRAF inhibition in melanoma, coupled with the emergence of acquired resistance, underscores the importance of rigorously validating quantitative biomarkers of treatment response in this and similar settings. Because constitutive activation of BRAF leads to proliferation in tumors, we explored 3'-deoxy-3'-(18)F-fluorothymidine ((18)F-FLT) PET to noninvasively quantify changes in tumor proliferation that are associated with pharmacologic inhibition of (V600E)BRAF downstream effectors and that precede changes in tumor volume.<br><br>Methods: Human colorectal cancer (CRC) cell lines expressing (V600E)BRAF were used to explore relationships between upregulation of p27 and phosphorylation of BRAF downstream effectors on small-molecule (V600E)BRAF inhibitor exposure. Athymic nude mice bearing (V600E)BRAF-expressing human CRC cell line xenografts were treated with a small-molecule (V600E)BRAF inhibitor (or vehicle) daily for 10 d. Predictive (18)F-FLT PET was conducted before changes in tumor volume occurred. Correlations were evaluated among PET, inhibition of phosphorylated MEK (p-MEK) and phosphorylated-ERK (p-ERK) by Western blot, tumor proliferation by histology, and small-molecule exposure by matrix-assisted laser desorption/ionization (MALDI) imaging mass spectrometry (IMS).<br><br>Results: Treatment of CRC cell lines with PLX4720 reduced proliferation associated with target inhibition and upregulation of p27. In vivo, PLX4720 treatment reduced (18)F-FLT uptake, but not (18)F-FDG uptake, in Lim2405 xenografts before quantifiable differences in xenograft volume. Reduced (18)F-FLT PET reflected a modest, yet significant, reduction of Ki67 immunoreactivity, inhibition of p-MEK and p-ERK, and elevated tumor cell p27 protein levels. Both (18)F-FLT PET and (18)F-FDG PET accurately reflected a lack of response in HT-29 xenografts, which MALDI imaging mass spectrometry suggested may have stemmed from limited PLX4720 exposure.<br><br>Conclusion: We used preclinical models of CRC to demonstrate (18)F-FLT PET as a sensitive predictor of response to (V600E)BRAF inhibitors. Because (18)F-FLT PET predicted reduced proliferation associated with attenuation of BRAF downstream effectors, yet (18)F-FDG PET did not, these data suggest that (18)F-FLT PET may represent an alternative to (18)F-FDG PET for quantifying clinical responses to BRAF inhibitors.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
=======
<li class="list-group-item"><strong>Eliot T. McKinley</strong>, R Adam Smith, Ping Zhao, Allie Fu, Samir A Saleh, Md Imam Uddin, M Kay Washington, Robert J Coffey, and H Charles Manning. (2013) "3'-Deoxy-3'-18F-fluorothymidine PET predicts response to (V600E)BRAF-targeted therapy in preclinical models of colorectal cancer." <em>J Nucl Med</em><br><br><details><summary>Abstract</summary><ul>Selective inhibition of oncogenic targets and associated signaling pathways forms the basis of personalized cancer medicine. The clinical success of (V600E)BRAF inhibition in melanoma, coupled with the emergence of acquired resistance, underscores the importance of rigorously validating quantitative biomarkers of treatment response in this and similar settings. Because constitutive activation of BRAF leads to proliferation in tumors, we explored 3'-deoxy-3'-(18)F-fluorothymidine ((18)F-FLT) PET to noninvasively quantify changes in tumor proliferation that are associated with pharmacologic inhibition of (V600E)BRAF downstream effectors and that precede changes in tumor volume.<br><br>Methods: Human colorectal cancer (CRC) cell lines expressing (V600E)BRAF were used to explore relationships between upregulation of p27 and phosphorylation of BRAF downstream effectors on small-molecule (V600E)BRAF inhibitor exposure. Athymic nude mice bearing (V600E)BRAF-expressing human CRC cell line xenografts were treated with a small-molecule (V600E)BRAF inhibitor (or vehicle) daily for 10 d. Predictive (18)F-FLT PET was conducted before changes in tumor volume occurred. Correlations were evaluated among PET, inhibition of phosphorylated MEK (p-MEK) and phosphorylated-ERK (p-ERK) by Western blot, tumor proliferation by histology, and small-molecule exposure by matrix-assisted laser desorption/ionization (MALDI) imaging mass spectrometry (IMS).<br><br>Results: Treatment of CRC cell lines with PLX4720 reduced proliferation associated with target inhibition and upregulation of p27. In vivo, PLX4720 treatment reduced (18)F-FLT uptake, but not (18)F-FDG uptake, in Lim2405 xenografts before quantifiable differences in xenograft volume. Reduced (18)F-FLT PET reflected a modest, yet significant, reduction of Ki67 immunoreactivity, inhibition of p-MEK and p-ERK, and elevated tumor cell p27 protein levels. Both (18)F-FLT PET and (18)F-FDG PET accurately reflected a lack of response in HT-29 xenografts, which MALDI imaging mass spectrometry suggested may have stemmed from limited PLX4720 exposure.<br><br>Conclusion: We used preclinical models of CRC to demonstrate (18)F-FLT PET as a sensitive predictor of response to (V600E)BRAF inhibitors. Because (18)F-FLT PET predicted reduced proliferation associated with attenuation of BRAF downstream effectors, yet (18)F-FDG PET did not, these data suggest that (18)F-FLT PET may represent an alternative to (18)F-FDG PET for quantifying clinical responses to BRAF inhibitors.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
>>>>>>> 9edb1aff55c5ced3256ae9d3470567a2ac9158f1
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/23341544/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.2967/jnumed.112.108456" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li></ul>
<h3 id="published-2012" class="anchored">2012</h3>

<<<<<<< HEAD
<ul class="list-group list-group-flush"><li class="list-group-item"><strong>Eliot T. McKinley</strong>, R. Adam Smith, Jarred P. Tanksley, M. Kay Washington, Ronald Walker, Robert J. Coffey, and H. Charles Manning. (2012) "[18F]FLT-PET to predict pharmacodynamic and clinical response to cetuximab therapy in Ménétrier's disease." <em>Ann Nucl Med</em><br><br><details><summary>Abstract</summary><ul>Molecular imaging biomarkers of proliferation hold great promise for quantifying response to personalized medicine. One such approach utilizes the positron emission tomography (PET) tracer 3'-deoxy-3'[18F]-fluorothymidine ([18F]FLT), an investigational agent whose uptake reflects thymidine salvage-dependent DNA synthesis. The goal of this study was to evaluate [18F]FLT-PET in the setting of Ménétrier's disease (MD), a rare, premalignant hyperproliferative disorder of the stomach treatable with cetuximab therapy. Over 15 months, a patient with confirmed MD underwent cetuximab therapy and was followed with sequential [18F]FLT-PET. For comparison to MD, an [18F]FLT-PET study was conducted in another patient to quantify uptake in a normal stomach. Prior to cetuximab therapy, stomach tissue in MD was easily visualized with [18F]FLT-PET, with pre-treatment uptake levels exceeding normal stomach uptake by approximately fourfold. Diminished [18F]FLT-PET in MD was observed following the initial and subsequent doses of cetuximab and correlated with clinical resolution of the disease. To our knowledge, this study reports the first clinical use of [18F]FLT-PET to assess proliferation in a premalignant disorder. We illustrate that the extent of MD involvement throughout the stomach could be easily visualized using [18F]FLT-PET, and that response to cetuximab could be followed quantitatively and non-invasively in sequential [18F]FLT-PET studies. Thus, [18F]FLT-PET appears to have potential to monitor response to treatment in this and potentially other hyperproliferative disorders.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
=======
<ul class="list-group list-group-flush"><li class="list-group-item"><strong>Eliot T. McKinley</strong>, R Adam Smith, Jarred P Tanksley, Mary Kay Washington, Ronald Walker, Robert J Coffey, and H Charles Manning. (2012) "[18F]FLT-PET to predict pharmacodynamic and clinical response to cetuximab therapy in Ménétrier's disease." <em>Ann Nucl Med</em><br><br><details><summary>Abstract</summary><ul>Molecular imaging biomarkers of proliferation hold great promise for quantifying response to personalized medicine. One such approach utilizes the positron emission tomography (PET) tracer 3'-deoxy-3'[18F]-fluorothymidine ([18F]FLT), an investigational agent whose uptake reflects thymidine salvage-dependent DNA synthesis. The goal of this study was to evaluate [18F]FLT-PET in the setting of Ménétrier's disease (MD), a rare, premalignant hyperproliferative disorder of the stomach treatable with cetuximab therapy. Over 15 months, a patient with confirmed MD underwent cetuximab therapy and was followed with sequential [18F]FLT-PET. For comparison to MD, an [18F]FLT-PET study was conducted in another patient to quantify uptake in a normal stomach. Prior to cetuximab therapy, stomach tissue in MD was easily visualized with [18F]FLT-PET, with pre-treatment uptake levels exceeding normal stomach uptake by approximately fourfold. Diminished [18F]FLT-PET in MD was observed following the initial and subsequent doses of cetuximab and correlated with clinical resolution of the disease. To our knowledge, this study reports the first clinical use of [18F]FLT-PET to assess proliferation in a premalignant disorder. We illustrate that the extent of MD involvement throughout the stomach could be easily visualized using [18F]FLT-PET, and that response to cetuximab could be followed quantitatively and non-invasively in sequential [18F]FLT-PET studies. Thus, [18F]FLT-PET appears to have potential to monitor response to treatment in this and potentially other hyperproliferative disorders.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
>>>>>>> 9edb1aff55c5ced3256ae9d3470567a2ac9158f1
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/22821337/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1007/s12149-012-0636-x" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li>
<<<<<<< HEAD
<li class="list-group-item">Dewei Tang, Matthew R. Hight, <strong>Eliot T. McKinley</strong>, Allie Fu, Jason R. Buck, R. Adam Smith, Mohammed Noor Tantawy, Todd E. Peterson, Daniel C. Colvin, M. Sib Ansari, Michael L. Nickels, and H. Charles Manning. (2012) "Quantitative preclinical imaging of TSPO expression in glioma using N,N-diethyl-2-(2-(4-(2-18F-fluoroethoxy)phenyl)-5,7-dimethylpyrazolo[1,5-a]pyrimidin-3-yl)acetamide." <em>J Nucl Med</em><br><br><details><summary>Abstract</summary><ul> </ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
=======
<li class="list-group-item">Dewei Tang, Matthew R Hight, <strong>Eliot T. McKinley</strong>, Allie Fu, Jason R Buck, R Adam Smith, Mohammed Noor Tantawy, Todd E Peterson, Daniel C Colvin, M Sib Ansari, Michael Nickels, and H Charles Manning. (2012) "Quantitative preclinical imaging of TSPO expression in glioma using N,N-diethyl-2-(2-(4-(2-18F-fluoroethoxy)phenyl)-5,7-dimethylpyrazolo[1,5-a]pyrimidin-3-yl)acetamide." <em>J Nucl Med</em><br><br><details><summary>Abstract</summary><ul> </ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
>>>>>>> 9edb1aff55c5ced3256ae9d3470567a2ac9158f1
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.2967/jnumed.111.095653" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li></ul>
<h3 id="published-2011" class="anchored">2011</h3>

<<<<<<< HEAD
<ul class="list-group list-group-flush"><li class="list-group-item">Joan T. Garrett, María Graciela Olivares, Cammie Rinehart, Nara D. Granja-Ingram, Violeta Sánchez, Anindita Chakrabarty, Bhuvanesh Dave, Rebecca S. Cook, William Pao, <strong>Eliot T. McKinley</strong>, H. Charles Manning, Jenny Chang, and Carlos L. Arteaga. (2011) "Transcriptional and posttranslational up-regulation of HER3 (ErbB3) compensates for inhibition of the HER2 tyrosine kinase." <em>Proc Natl Acad Sci U S A</em><br><br><details><summary>Abstract</summary><ul>Sustained and complete inhibition of HER3 and its output to PI3K/Akt are required for the optimal antitumor effect of therapeutic inhibitors of the HER2 oncogene. Here, we show that, after inhibition of the HER2 tyrosine kinase with lapatinib, there is PI3K/Akt and FoxO3a-dependent up-regulation of HER3 mRNA and protein. Up-regulated HER3 was then phosphorylated by residual HER2 activity, thus partially maintaining P-Akt and limiting the antitumor action of lapatinib. Inhibition of HER3 with siRNA or a neutralizing HER3 antibody sensitized HER2+ breast cancer cells and xenografts to lapatinib both in vitro and in vivo. Combined blockade of HER2 and HER3 inhibited pharmacodynamic biomarkers of PI3K/Akt activity more effectively than each inhibitor alone. These results suggest that because of HER3-mediated compensation, current clinical inhibitors of HER2 and PI3K/Akt will not block the PI3K pathway completely. They also suggest that therapeutic inhibitors of HER3 should be used in combination with HER2 inhibitors and PI3K pathway inhibitors in patients with HER2- and PI3K-dependent cancers.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
=======
<ul class="list-group list-group-flush"><li class="list-group-item">Joan T Garrett, María Graciela Olivares, Cammie Rinehart, Nara D Granja-Ingram, Violeta Sánchez, Anindita Chakrabarty, Bhuvanesh Dave, Rebecca S Cook, William Pao, <strong>Eliot T. McKinley</strong>, H C Manning, Jenny Chang, and Carlos L Arteaga. (2011) "Transcriptional and posttranslational up-regulation of HER3 (ErbB3) compensates for inhibition of the HER2 tyrosine kinase." <em>Proc Natl Acad Sci U S A</em><br><br><details><summary>Abstract</summary><ul>Sustained and complete inhibition of HER3 and its output to PI3K/Akt are required for the optimal antitumor effect of therapeutic inhibitors of the HER2 oncogene. Here, we show that, after inhibition of the HER2 tyrosine kinase with lapatinib, there is PI3K/Akt and FoxO3a-dependent up-regulation of HER3 mRNA and protein. Up-regulated HER3 was then phosphorylated by residual HER2 activity, thus partially maintaining P-Akt and limiting the antitumor action of lapatinib. Inhibition of HER3 with siRNA or a neutralizing HER3 antibody sensitized HER2+ breast cancer cells and xenografts to lapatinib both in vitro and in vivo. Combined blockade of HER2 and HER3 inhibited pharmacodynamic biomarkers of PI3K/Akt activity more effectively than each inhibitor alone. These results suggest that because of HER3-mediated compensation, current clinical inhibitors of HER2 and PI3K/Akt will not block the PI3K pathway completely. They also suggest that therapeutic inhibitors of HER3 should be used in combination with HER2 inhibitors and PI3K pathway inhibitors in patients with HER2- and PI3K-dependent cancers.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
>>>>>>> 9edb1aff55c5ced3256ae9d3470567a2ac9158f1
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/21385943/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1073/pnas.1016140108" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li>
<<<<<<< HEAD
<li class="list-group-item"><strong>Eliot T. McKinley</strong>, Joseph E Bugaj, Ping Zhao, Saffet Guleryuz, Christine Mantis, Prafulla C Gokhale, Robert Wild, and H. Charles Manning. (2011) "18FDG-PET predicts pharmacodynamic response to OSI-906, a dual IGF-1R/IR inhibitor, in preclinical mouse models of lung cancer." <em>Clin Cancer Res</em><br><br><details><summary>Abstract</summary><ul>Purpose: To evaluate 2-deoxy-2-[(18)F]fluoro-d-glucose positron emission tomography imaging ((18)FDG-PET) as a predictive, noninvasive, pharmacodynamic (PD) biomarker of response following administration of a small-molecule insulin-like growth factor-1 receptor and insulin receptor (IGF-1R/IR) inhibitor, OSI-906.<br><br>Experimental design: In vitro uptake studies of (3)H-2-deoxy glucose following OSI-906 exposure were conducted evaluating correlation of dose with inhibition of IGF-1R/IR as well as markers of downstream pathways and glucose metabolism. Similarly, in vivo PD effects were evaluated in human tumor cell line xenografts propagated in athymic nude mice by (18)FDG-PET at 2, 4, and 24 hours following a single treatment of OSI-906 for the correlation of inhibition of receptor targets and downstream markers.<br><br>Results: Uptake of (3)H-2-deoxy glucose and (18)FDG was significantly diminished following OSI-906 exposure in sensitive tumor cells and subcutaneous xenografts (NCI-H292) but not in an insensitive model lacking IGF-1R expression (NCI-H441). Diminished PD (18)FDG-PET, collected immediately following the initial treatment agreed with inhibition of pIGF-1R/pIR, reduced PI3K (phosphoinositide 3-kinase) and MAPK (mitogen activated protein kinase) pathway activity, and predicted tumor growth arrest as measured by high-resolution ultrasound imaging.<br><br>Conclusion: (18)FDG-PET seems to serve as a rapid, noninvasive PD marker of IGF-1R/IR inhibition following a single dose of OSI-906 and should be explored clinically as a predictive clinical biomarker in patients undergoing IGF-1R/IR-directed cancer therapy.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
=======
<li class="list-group-item"><strong>Eliot T. McKinley</strong>, Joseph E Bugaj, Ping Zhao, Saffet Guleryuz, Christine Mantis, Prafulla C Gokhale, Robert Wild, and H Charles Manning. (2011) "18FDG-PET predicts pharmacodynamic response to OSI-906, a dual IGF-1R/IR inhibitor, in preclinical mouse models of lung cancer." <em>Clin Cancer Res</em><br><br><details><summary>Abstract</summary><ul>Purpose: To evaluate 2-deoxy-2-[(18)F]fluoro-d-glucose positron emission tomography imaging ((18)FDG-PET) as a predictive, noninvasive, pharmacodynamic (PD) biomarker of response following administration of a small-molecule insulin-like growth factor-1 receptor and insulin receptor (IGF-1R/IR) inhibitor, OSI-906.<br><br>Experimental design: In vitro uptake studies of (3)H-2-deoxy glucose following OSI-906 exposure were conducted evaluating correlation of dose with inhibition of IGF-1R/IR as well as markers of downstream pathways and glucose metabolism. Similarly, in vivo PD effects were evaluated in human tumor cell line xenografts propagated in athymic nude mice by (18)FDG-PET at 2, 4, and 24 hours following a single treatment of OSI-906 for the correlation of inhibition of receptor targets and downstream markers.<br><br>Results: Uptake of (3)H-2-deoxy glucose and (18)FDG was significantly diminished following OSI-906 exposure in sensitive tumor cells and subcutaneous xenografts (NCI-H292) but not in an insensitive model lacking IGF-1R expression (NCI-H441). Diminished PD (18)FDG-PET, collected immediately following the initial treatment agreed with inhibition of pIGF-1R/pIR, reduced PI3K (phosphoinositide 3-kinase) and MAPK (mitogen activated protein kinase) pathway activity, and predicted tumor growth arrest as measured by high-resolution ultrasound imaging.<br><br>Conclusion: (18)FDG-PET seems to serve as a rapid, noninvasive PD marker of IGF-1R/IR inhibition following a single dose of OSI-906 and should be explored clinically as a predictive clinical biomarker in patients undergoing IGF-1R/IR-directed cancer therapy.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
>>>>>>> 9edb1aff55c5ced3256ae9d3470567a2ac9158f1
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/21257723/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1158/1078-0432.ccr-10-2274" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li>
<<<<<<< HEAD
<li class="list-group-item">Matthew R. Hight, Donald D. Nolting, <strong>Eliot T. McKinley</strong>, Adam D. Lander, Shelby K. Wyatt, Mark Gonyea, Ping Zhao, and H. Charles Manning. (2011) "Multispectral fluorescence imaging to assess pH in biological specimens." <em>J Biomed Opt</em><br><br><details><summary>Abstract</summary><ul>Simple, quantitative assays to measure pH in tissue could improve the study of complicated biological processes and diseases such as cancer. We evaluated multispectral fluorescence imaging (MSFI) to quantify extracellular pH (pHe) in dye-perfused, surgically-resected tumor specimens with commercially available instrumentation. Utilizing a water-soluble organic dye with pH-dependent fluorescence emission (SNARF-4F), we used standard fluorimetry to quantitatively assess the emission properties of the dye as a function of pH. By conducting these studies within the spectroscopic constraints imposed by the appropriate imaging filter set supplied with the imaging system, we determined that correction of the fluorescence emission of deprotonated dye was necessary for accurate determination of pH due to suboptimal excitation. Subsequently, employing a fluorimetry-derived correction factor (CF), MSFI data sets of aqueous dye solutions and tissuelike phantoms could be spectrally unmixed to accurately quantify equilibrium concentrations of protonated (HA) and deprotonated (A-) dye and thus determine solution pH. Finally, we explored the feasibility of MSFI for high-resolution pHe mapping of human colorectal cancer cell-line xenografts. Data presented suggest that MSFI is suitable for quantitative determination of pHe in ex vivo dye-perfused tissue, potentially enabling measurement of pH across a variety of preclinical models of disease.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
=======
<li class="list-group-item">Matthew R Hight, Donald D Nolting, Eliot T McKinley, Adam D Lander, Shelby K Wyatt, Mark Gonyea, Ping Zhao, and H Charles Manning. (2011) "Multispectral fluorescence imaging to assess pH in biological specimens." <em>J Biomed Opt</em><br><br><details><summary>Abstract</summary><ul>Simple, quantitative assays to measure pH in tissue could improve the study of complicated biological processes and diseases such as cancer. We evaluated multispectral fluorescence imaging (MSFI) to quantify extracellular pH (pHe) in dye-perfused, surgically-resected tumor specimens with commercially available instrumentation. Utilizing a water-soluble organic dye with pH-dependent fluorescence emission (SNARF-4F), we used standard fluorimetry to quantitatively assess the emission properties of the dye as a function of pH. By conducting these studies within the spectroscopic constraints imposed by the appropriate imaging filter set supplied with the imaging system, we determined that correction of the fluorescence emission of deprotonated dye was necessary for accurate determination of pH due to suboptimal excitation. Subsequently, employing a fluorimetry-derived correction factor (CF), MSFI data sets of aqueous dye solutions and tissuelike phantoms could be spectrally unmixed to accurately quantify equilibrium concentrations of protonated (HA) and deprotonated (A-) dye and thus determine solution pH. Finally, we explored the feasibility of MSFI for high-resolution pHe mapping of human colorectal cancer cell-line xenografts. Data presented suggest that MSFI is suitable for quantitative determination of pHe in ex vivo dye-perfused tissue, potentially enabling measurement of pH across a variety of preclinical models of disease.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
>>>>>>> 9edb1aff55c5ced3256ae9d3470567a2ac9158f1
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/21280913/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1117/1.3533264" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li>
<<<<<<< HEAD
<li class="list-group-item">Jason R. Buck, <strong>Eliot T. McKinley</strong>, Matthew R. Hight, Allie Fu, Dewei Tang, R. Adam Smith, Mohammed Noor Tantawy, Todd E. Peterson, Daniel C. Colvin, Mohammed Sib Ansari, Ronald M. Baldwin, Ping Zhao, Saffet Guleryuz, and H. Charles Manning. (2011) "Quantitative, preclinical PET of translocator protein expression in glioma using 18F-N-fluoroacetyl-N-(2,5-dimethoxybenzyl)-2-phenoxyaniline." <em>J Nucl Med</em><br><br><details><summary>Abstract</summary><ul>Translocator protein (TSPO), also referred to as peripheral benzodiazepine receptor (PBR), is a crucial 18-kDa outer mitochondrial membrane protein involved in numerous cellular functions, including the regulation of cholesterol metabolism, steroidogenesis, and apoptosis. Elevated expression of TSPO in oncology correlates with disease progression and poor survival, suggesting that molecular probes capable of assaying TSPO levels may have potential as cancer imaging biomarkers. In preclinical PET studies, we characterized a high-affinity aryloxyanilide-based TSPO imaging ligand, 18F-N-fluoroacetyl-N-(2,5-dimethoxybenzyl)-2-phenoxyaniline (18F-PBR06), as a candidate probe for the quantitative assessment of TSPO expression in glioma.<br><br>Methods: Glioma-bearing rats were imaged with 18F-PBR06 in a small-animal PET system. Dynamic images were acquired simultaneously on injection of 18F-PBR06 (70-100 MBq/0.2 mL). Over the course of scanning, arterial blood was collected to derive the input function, with high-performance liquid chromatography radiometabolite analysis performed on selected samples for arterial input function correction. Compartmental modeling of the PET data was performed using the corrected arterial input function. Specific tumor cell binding of PBR06 was evaluated by radioligand displacement of 3H-PK 11195 with PBR06 in vitro and by displacement of 18F-PBR06 with excess PBR06 in vivo. Immediately after imaging, tumor tissue and adjacent healthy brain were harvested for assay of TSPO protein levels by Western blotting and immunohistochemistry.<br><br>Results: 18F-PBR06 was found to preferentially accumulate in tumors, with modest uptake in the contralateral brain, facilitating excellent contrast between tumor and adjacent tissue. Infusion with PBR06 (10 mg/kg) displaced 18F-PBR06 binding by approximately 75%. The accumulation of 18F-PBR06 in tumor tissues and adjacent brain agreed with the ex vivo assay of TSPO protein levels by Western blotting and quantitative immunohistochemistry.<br><br>Conclusion: These preclinical studies illustrate that 18F-PBR06 is a promising tracer for visualization of TSPO-expressing tumors. Importantly, the close correlation between 18F-PBR06 uptake and TSPO expression in tumors and normal tissues, coupled with the high degree of displaceable binding from both tumors and the normal brain, represents a significant improvement over other TSPO imaging ligands previously evaluated in glioma. These data suggest the potential of 18F-PBR06 to elucidate the role of TSPO in oncology, as well as its potential development as a cancer imaging biomarker.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
=======
<li class="list-group-item">Jason R Buck, <strong>Eliot T. McKinley</strong>, Matthew R Hight, Allie Fu, Dewei Tang, Ralph Adam Smith, Mohammed Noor Tantawy, Todd E Peterson, Daniel Colvin, Mohammed Sib Ansari, Ronald M Baldwin, Ping Zhao, Saffet Guleryuz, and H Charles Manning. (2011) "Quantitative, preclinical PET of translocator protein expression in glioma using 18F-N-fluoroacetyl-N-(2,5-dimethoxybenzyl)-2-phenoxyaniline." <em>J Nucl Med</em><br><br><details><summary>Abstract</summary><ul>Translocator protein (TSPO), also referred to as peripheral benzodiazepine receptor (PBR), is a crucial 18-kDa outer mitochondrial membrane protein involved in numerous cellular functions, including the regulation of cholesterol metabolism, steroidogenesis, and apoptosis. Elevated expression of TSPO in oncology correlates with disease progression and poor survival, suggesting that molecular probes capable of assaying TSPO levels may have potential as cancer imaging biomarkers. In preclinical PET studies, we characterized a high-affinity aryloxyanilide-based TSPO imaging ligand, 18F-N-fluoroacetyl-N-(2,5-dimethoxybenzyl)-2-phenoxyaniline (18F-PBR06), as a candidate probe for the quantitative assessment of TSPO expression in glioma.<br><br>Methods: Glioma-bearing rats were imaged with 18F-PBR06 in a small-animal PET system. Dynamic images were acquired simultaneously on injection of 18F-PBR06 (70-100 MBq/0.2 mL). Over the course of scanning, arterial blood was collected to derive the input function, with high-performance liquid chromatography radiometabolite analysis performed on selected samples for arterial input function correction. Compartmental modeling of the PET data was performed using the corrected arterial input function. Specific tumor cell binding of PBR06 was evaluated by radioligand displacement of 3H-PK 11195 with PBR06 in vitro and by displacement of 18F-PBR06 with excess PBR06 in vivo. Immediately after imaging, tumor tissue and adjacent healthy brain were harvested for assay of TSPO protein levels by Western blotting and immunohistochemistry.<br><br>Results: 18F-PBR06 was found to preferentially accumulate in tumors, with modest uptake in the contralateral brain, facilitating excellent contrast between tumor and adjacent tissue. Infusion with PBR06 (10 mg/kg) displaced 18F-PBR06 binding by approximately 75%. The accumulation of 18F-PBR06 in tumor tissues and adjacent brain agreed with the ex vivo assay of TSPO protein levels by Western blotting and quantitative immunohistochemistry.<br><br>Conclusion: These preclinical studies illustrate that 18F-PBR06 is a promising tracer for visualization of TSPO-expressing tumors. Importantly, the close correlation between 18F-PBR06 uptake and TSPO expression in tumors and normal tissues, coupled with the high degree of displaceable binding from both tumors and the normal brain, represents a significant improvement over other TSPO imaging ligands previously evaluated in glioma. These data suggest the potential of 18F-PBR06 to elucidate the role of TSPO in oncology, as well as its potential development as a cancer imaging biomarker.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
>>>>>>> 9edb1aff55c5ced3256ae9d3470567a2ac9158f1
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/21149488/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.2967/jnumed.110.081703" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li></ul>
<h3 id="published-2010" class="anchored">2010</h3>

<ul class="list-group list-group-flush"><li class="list-group-item">Gregory D. Ayers, <strong>Eliot T. McKinley</strong>, Ping Zhao, Jordan M. Fritz, Rebecca E. Metry, Brenton C. Deal, Katrina M. Adlerz, Robert J. Coffey, and H. Charles Manning. (2010) "Volume of preclinical xenograft tumors is more accurately assessed by ultrasound imaging than manual caliper measurements." <em>J Ultrasound Med</em><br><br><details><summary>Abstract</summary><ul>Objective: The volume of subcutaneous xenograft tumors is an important metric of disease progression and response to therapy in preclinical drug development. Noninvasive imaging technologies suitable for measuring xenograft volume are increasingly available, yet manual calipers, which are susceptible to inaccuracy and bias, are routinely used. The goal of this study was to quantify and compare the accuracy, precision, and inter-rater variability of xenograft tumor volume assessment by caliper measurements and ultrasound imaging.<br><br>Methods: Subcutaneous xenograft tumors derived from human colorectal cancer cell lines (DLD1 and SW620) were generated in athymic nude mice. Experienced independent reviewers segmented 3-dimensional ultrasound data sets and collected manual caliper measurements resulting in tumor volumes. Imaging- and caliper-derived volumes were compared with the tumor mass, the reference standard, determined after resection. Bias, precision, and inter-rater differences were estimated for each mouse among reviewers. Bootstrapping was used to estimate mean and confidence intervals of variance components, intraclass correlation coefficients (ICCs), and confidence intervals for each source of variation.<br><br>Results: The average deviation from the true volume and inter-rater differences were significantly lower for ultrasound volumes compared with caliper volumes (P = .0005 and .001, respectively). Reviewer ICCs for ultrasound and caliper measurements were similarly low (1%), yet caliper volume variance was 1.3-fold higher than for ultrasound.<br><br>Conclusions: Ultrasound imaging more accurately, precisely, and reproducibly reflects xenograft tumor volume than caliper measurements. These data suggest that preclinical studies using the xenograft burden as a surrogate end point measured by ultrasound imaging require up to 30% fewer animals to reach statistical significance compared with analogous studies using caliper measurements.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/20498463/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.7863%2Fjum.2010.29.6.891" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li></ul>
<h3 id="published-2009" class="anchored">2009</h3>
<<<<<<< HEAD
<ul class="list-group list-group-flush"><li class="list-group-item">Chirayu Shah, Todd W. Miller, Shelby K. Wyatt, <strong>Eliot T. McKinley</strong>, Maria Graciela Olivares, Violeta Sánchez, Donald D. Nolting, Jason R. Buck, Ping Zhao, M. Sib Ansari, Ronald M. Baldwin, John C. Gore, Rachel Schiff, Carlos L. Arteaga, and H. Charles Manning. (2009) "Imaging biomarkers predict response to anti-HER2 (ErbB2) therapy in preclinical models of breast cancer." <em>Clin Cancer Res</em><br><br><details><summary>Abstract</summary><ul>Purpose: To evaluate noninvasive imaging methods as predictive biomarkers of response to trastuzumab in mouse models of HER2-overexpressing breast cancer. The correlation between tumor regression and molecular imaging of apoptosis, glucose metabolism, and cellular proliferation was evaluated longitudinally in responding and nonresponding tumor-bearing cohorts.<br><br>Experimental design: Mammary tumors from MMTV/HER2 transgenic female mice were transplanted into syngeneic female mice. BT474 human breast carcinoma cell line xenografts were grown in athymic nude mice. Tumor cell apoptosis (NIR700-Annexin V accumulation), glucose metabolism [2-deoxy-2-[18F]fluoro-d-glucose positron emission tomography ([18F]FDG-PET)], and proliferation [3'-[18F]fluoro-3'-deoxythymidine-PET ([18F]FLT-PET)] were evaluated throughout a biweekly trastuzumab regimen. Imaging metrics were validated by direct measurement of tumor size and immunohistochemical analysis of cleaved caspase-3, phosphorylated AKT, and Ki67.<br><br>Results: NIR700-Annexin V accumulated significantly in trastuzumab-treated MMTV/HER2 and BT474 tumors that ultimately regressed but not in nonresponding or vehicle-treated tumors. Uptake of [18F]FDG was not affected by trastuzumab treatment in MMTV/HER2 or BT474 tumors. [18F]FLT-PET imaging predicted trastuzumab response in BT474 tumors but not in MMTV/HER2 tumors, which exhibited modest uptake of [18F]FLT. Close agreement was observed between imaging metrics and immunohistochemical analysis.<br><br>Conclusions: Molecular imaging of apoptosis accurately predicts trastuzumab-induced regression of HER2+ tumors and may warrant clinical exploration to predict early response to neoadjuvant trastuzumab. Trastuzumab does not seem to alter glucose metabolism substantially enough to afford [18F]FDG-PET significant predictive value in this setting. Although promising in one preclinical model, further studies are required to determine the overall value of [18F]FLT-PET as a biomarker of response to trastuzumab in HER2+ breast cancer.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
=======
<ul class="list-group list-group-flush"><li class="list-group-item">Chirayu Shah, Todd W Miller, Shelby K Wyatt, <strong>Eliot T. McKinley</strong>, Maria Graciela Olivares, Violeta Sanchez, Donald D Nolting, Jason R Buck, Ping Zhao, M Sib Ansari, Ronald M Baldwin, John C Gore, Rachel Schiff, Carlos L Arteaga, and H Charles Manning. (2009) "Imaging biomarkers predict response to anti-HER2 (ErbB2) therapy in preclinical models of breast cancer." <em>Clin Cancer Res</em><br><br><details><summary>Abstract</summary><ul>Purpose: To evaluate noninvasive imaging methods as predictive biomarkers of response to trastuzumab in mouse models of HER2-overexpressing breast cancer. The correlation between tumor regression and molecular imaging of apoptosis, glucose metabolism, and cellular proliferation was evaluated longitudinally in responding and nonresponding tumor-bearing cohorts.<br><br>Experimental design: Mammary tumors from MMTV/HER2 transgenic female mice were transplanted into syngeneic female mice. BT474 human breast carcinoma cell line xenografts were grown in athymic nude mice. Tumor cell apoptosis (NIR700-Annexin V accumulation), glucose metabolism [2-deoxy-2-[18F]fluoro-d-glucose positron emission tomography ([18F]FDG-PET)], and proliferation [3'-[18F]fluoro-3'-deoxythymidine-PET ([18F]FLT-PET)] were evaluated throughout a biweekly trastuzumab regimen. Imaging metrics were validated by direct measurement of tumor size and immunohistochemical analysis of cleaved caspase-3, phosphorylated AKT, and Ki67.<br><br>Results: NIR700-Annexin V accumulated significantly in trastuzumab-treated MMTV/HER2 and BT474 tumors that ultimately regressed but not in nonresponding or vehicle-treated tumors. Uptake of [18F]FDG was not affected by trastuzumab treatment in MMTV/HER2 or BT474 tumors. [18F]FLT-PET imaging predicted trastuzumab response in BT474 tumors but not in MMTV/HER2 tumors, which exhibited modest uptake of [18F]FLT. Close agreement was observed between imaging metrics and immunohistochemical analysis.<br><br>Conclusions: Molecular imaging of apoptosis accurately predicts trastuzumab-induced regression of HER2+ tumors and may warrant clinical exploration to predict early response to neoadjuvant trastuzumab. Trastuzumab does not seem to alter glucose metabolism substantially enough to afford [18F]FDG-PET significant predictive value in this setting. Although promising in one preclinical model, further studies are required to determine the overall value of [18F]FLT-PET as a biomarker of response to trastuzumab in HER2+ breast cancer.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
>>>>>>> 9edb1aff55c5ced3256ae9d3470567a2ac9158f1
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/19584166/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1158/1078-0432.ccr-08-2635" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li></ul>
<h3 id="published-2008" class="anchored">2008</h3>
<<<<<<< HEAD
<ul class="list-group list-group-flush"><li class="list-group-item">H. Charles Manning, Adam Lander, <strong>Eliot T. McKinley</strong>, and Nathan J. Mutic. (2008) "Accelerating the development of novel molecular imaging probes: a role for high-throughput screening." <em>J Nucl Med</em><br><br><details><summary>Abstract</summary><ul>Molecular imaging is a rapidly emerging research tool and clinical discipline aimed at noninvasive, quantitative visualization of in vivo molecular processes occurring at cellular and subcellular levels. At present, advancement of the molecular imaging field is driven by the development of improved imaging hardware for use in preclinical and clinical settings, the identification and validation of new, biologically relevant imaging targets, and the development of improved imaging probes derived from novel chemistries. Of these 3 essential facets, which comprise a majority of current molecular imaging research, hardware development and novel target discovery significantly outpace the development and clinical advancement of new molecular imaging probes, particularly with respect to cancer imaging.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
=======
<ul class="list-group list-group-flush"><li class="list-group-item">H Charles Manning, Adam Lander, <strong>Eliot T. McKinley</strong>, and Nathan J Mutic. (2008) "Accelerating the development of novel molecular imaging probes: a role for high-throughput screening." <em>J Nucl Med</em><br><br><details><summary>Abstract</summary><ul>Molecular imaging is a rapidly emerging research tool and clinical discipline aimed at noninvasive, quantitative visualization of in vivo molecular processes occurring at cellular and subcellular levels. At present, advancement of the molecular imaging field is driven by the development of improved imaging hardware for use in preclinical and clinical settings, the identification and validation of new, biologically relevant imaging targets, and the development of improved imaging probes derived from novel chemistries. Of these 3 essential facets, which comprise a majority of current molecular imaging research, hardware development and novel target discovery significantly outpace the development and clinical advancement of new molecular imaging probes, particularly with respect to cancer imaging.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
>>>>>>> 9edb1aff55c5ced3256ae9d3470567a2ac9158f1
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/18703594/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.2967/jnumed.108.053009" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li>
<li class="list-group-item">H. Charles Manning, Nipun B. Merchant, A. Coe Foutch, John M. Virostko, Shelby K. Wyatt, Chirayu Shah, <strong>Eliot T. McKinley</strong>, Jingping Xie, Nathan J. Mutic, M. Kay Washington, Bonnie LaFleur, Mohammed Noor Tantawy, Todd E. Peterson, M. Sib Ansari, Ronald M. Baldwin, Mace L. Rothenberg, Darryl J. Bornhop, John C. Gore, and Robert J. Coffey. (2008) "Molecular imaging of therapeutic response to epidermal growth factor receptor blockade in colorectal cancer." <em>Clin Cancer Res</em><br><br><details><summary>Abstract</summary><ul>Purpose: To evaluate noninvasive molecular imaging methods as correlative biomarkers of therapeutic efficacy of cetuximab in human colorectal cancer cell line xenografts grown in athymic nude mice. The correlation between molecular imaging and immunohistochemical analysis to quantify epidermal growth factor (EGF) binding, apoptosis, and proliferation was evaluated in treated and untreated tumor-bearing cohorts.<br><br>Experimental design: Optical imaging probes targeting EGF receptor (EGFR) expression (NIR800-EGF) and apoptosis (NIR700-Annexin V) were synthesized and evaluated in vitro and in vivo. Proliferation was assessed by 3'-[18F]fluoro-3'-deoxythymidine ([18F]FLT) positron emission tomography. Assessment of inhibition of EGFR signaling by cetuximab was accomplished by concomitant imaging of NIR800-EGF, NIR700-Annexin V, and [18F]FLT in cetuximab-sensitive (DiFi) and insensitive (HCT-116) human colorectal cancer cell line xenografts. Imaging results were validated by measurement of tumor size and immunohistochemical analysis of total and phosphorylated EGFR, caspase-3, and Ki-67 immediately following in vivo imaging.<br><br>Results: NIR800-EGF accumulation in tumors reflected relative EGFR expression and EGFR occupancy by cetuximab. NIR700-Annexin V accumulation correlated with cetuximab-induced apoptosis as assessed by immunohistochemical staining of caspase-3. No significant difference in tumor proliferation was noted between treated and untreated animals by [18F]FLT positron emission tomography or Ki-67 immunohistochemistry.<br><br>Conclusions: Molecular imaging can accurately assess EGF binding, proliferation, and apoptosis in human colorectal cancer xenografts. These imaging approaches may prove useful for serial, noninvasive monitoring of the biological effects of EGFR inhibition in preclinical studies. It is anticipated that these assays can be adapted for clinical use.</ul></details><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://pubmed.ncbi.nlm.nih.gov/19010858/" target="_blank" rel="noopener noreferrer">
        <i class="ai ai-pubmed" role="img" aria-label="PubMed"></i>
        PubMed
    </a><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <a class="btn btn-outline-dark btn-sm" ,="" href="https://doi.org/10.1158/1078-0432.ccr-08-0239" target="_blank" rel="noopener noreferrer">
        <i class="bi bi-journal-check" role="img" aria-label="Published"></i>
        Published
    </a></li></ul>



</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    setTimeout(function() {
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      let href = ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const cites = ref.parentNode.getAttribute('data-cites').split(' ');
    tippyHover(ref, function() {
      var popup = window.document.createElement('div');
      cites.forEach(function(cite) {
        var citeDiv = window.document.createElement('div');
        citeDiv.classList.add('hanging-indent');
        citeDiv.classList.add('csl-entry');
        var biblioDiv = window.document.getElementById('ref-' + cite);
        if (biblioDiv) {
          citeDiv.innerHTML = biblioDiv.innerHTML;
        }
        popup.appendChild(citeDiv);
      });
      return popup.innerHTML;
    });
  }
});
</script>
</div> <!-- /content -->



</body></html>